# Experimental Procedures

## *Drosophila* strains and genetic crosses

*Canton-S* was used as the wild-type (WT) strain.
The following mutants were used: *emc^tink^*, a null allele induced by ethyl methanesulfonate in this study;
*emc^AP6^*, an amorphic allele [@Ellis1994] \(Bloomington \#36544\);
*emc^2^*, a hypomorphic allele [@Ellis1990] \(Kyoto DGRC \#101588\);
*da^10^*, an amorphic allele [@Wulbeck1994] \(Bloomington \#5531\);
and *Myo31DF^K2^*, an amorphic allele [@Petzoldt2012].
The following UAS lines were used: *UAS-emc::GFP* [@Popova2011];
*UAS-da* [@Giebel1997] \(Bloomington \#51669\);
*UAS-myrGFP*;
and *UAS-Myo31DF::mRFP* [@Hatori2014].
The following *Gal4*-driver lines were used: *NP2432*, a hindgut epithelium-specific driver [@Hayashi2002] \(Kyoto DGRC \#104201\);
*byn-Gal4*, a hindgut epithelium-specific driver;
and *hand-Gal4*, a visceral muscle cell-specific driver [@Kuroda2012].
To express *Cre* in *Drosophila*, *y^1^, w^67c23^ P{y^+mDint2^=Crey}1b* was used to generate mosaic tissues [@Nakazawa2012].

Mutations on the first and second chromosome were balanced with *FM7c, P{ftz/lacC}YH1* and *CyO, P{en1}wg^en11^*, respectively.
Mutations on the third chromosome were balanced with _TM3, P{ftz-lacZ.ry^+^}TM3, Sb^1^ ry^\*^_, _TM6B, P{iab-2(1.7)lacZ}6B, Tb^1^_, or _TM3, P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb^1^ Ser^1^_.
All genetic crosses were performed at 25 °C on a standard *Drosophila* culture medium.

## Preparation of *Drosophila* embryos

*Drosophila* embryos were collected and dechorionated in 50% haiter (Kao).
The dechorionated embryos were fixed by 6% of paraformaldehyde in PBS (130 mM NaCl, 7 mM Na~2~HPO~4~, and 3 mM NaH~2~PO~4~) for 30 minutes.
After the fixation, the vitelline membrane of the embryos was removed by washing in 100% methanol or by the manual dissection.
To preserve the fixed embryos, they were kept in 100 % methanol at -20 °C.

## X-Gal Staining of embryos

To select the homozygous embryos of mutants, X-gal staining was performed [@Sullivan2000].
The embryos expressing *lacZ* were stained with X-Gal (5-bromo-4-chloro-3-indolyl-β-d-galactopyranoside) in X-Gal buffer (3 mM K~3~Fe\[CN\]~6~, 3 mM K~4~Fe\[CN\]~6~, 0.9 mM MgCl~2~, and 0.1 % (w/v) Tween-20 in PBS) at 37 °C for 1 hr.

## Immunolabeling of embryos

Fixed embryos were treated with 2 % Block Ace (Yukijirushi) in PBT (130 mM NaCl, 7 mM Na~2~HPO~4~, 3 mM NaH~2~PO~4~, and 0.1 % (w/v) Triton-X-100).
After washing the embryos with PBT, they were incubated with the primary antibodies solution in Can Get Signal Solution B (TOYOBO), IMMUNO SHOT Strong (Cosmo Bio), or 2% Block Ace (Yukijirushi) in PBT in 2 mL tubes (Eppendorf) at room temperature for 3 hr or at 4 °C over night.
The embryos were washed with PBT, and incubated with the secondary antibodies solution at room temperature for 3 hr or at 4 °C over night.
After washing the embryos, they were mounted with 60% glycerol.

The following primary antibodies were used at the dilution indicated:
mouse anti-phosphotyrosine antibody (PY20, the marker for cell membrane) (1:1000, BD Biosciences),
rat anti-*D*E-Cadherin (Cad) antibody (1:100, DSHB),
rabbit anti-GFP antibody (1:500, MBL),
rabbit anti-RFP antibody (1:1000, MBL),
mouse anti-Wingless antibody (1:1000, DSHB),
mouse anti-Engrailed (1:1000, DSHB),
mouse anti-Crumbs (1:1000, DSHB),
chicken anti-β-galactosidase antibody (1:500, Abcam),
and mouse anti-Lamin C antibody (1:1000, DSHB).

The following secondary antibodies were used at the dilution indicated: anti-mouse IgG-Cy3 (1:1000, Jackson ImmunoResearch),
anti-mouse IgG-Alexa Fluor 488 (1:1000, Molecular Probes),
anti-mouse IgG-Cy5 (1:1000, Jackson ImmunoResearch),
anti-rabbit IgG-Alexa Fluor 488 (1:1000, Molecular Probes),
anti-rabbit IgG-Cy3 (1:1000, Jackson ImmunoResearch),
anti-rabbit IgG-Cy5 (1:1000, Jackson ImmunoResearch),
anti-rat IgG-Cy3 (1:1000, Jackson ImmunoResearch),
anti-rat IgG-Alexa488 (1:1000, Jackson ImmunoResearch),
anti-rat IgG-Cy5 (1:1000, Jackson ImmunoResearch),
and anti-chicken IgY-Alexa488 (1:1000, Jackson ImmunoResearch).
To detect F-actin, rhodamine phalloidin was used (1:250, Thermo Fisher Scientific).

To evaluate LR defects of the hindgut, embryos were mounted in 50% glycerol and observed under a differential interference microscope (Axioskop 2 plus, Zeiss).
To obtain images of the hindgut, embryos were treated with a tissue-clearing reagent (methyl salicylate), and examined by confocal microscopy (LSM880, Carl Zeiss).
The images were processed using Fiji software, Adobe Photoshop CS3 (Adobe Systems), or Adobe Illustrator CS3 (Adobe Systems).

## DNA amplification by PCR and sequencing

Genomic DNA (gDNA) were collected from 30 embryos of wild-type or *emc^tink^* homozygous embryos using the standard protocol described previously [@Sullivan2000].
To amplify the *emc* locus, I performed polymerase chain reaction (PCR) using ExTaq (Takara), according to the product manual.
The primers used to amplify *emc* were 5'-TCGAACATCGACAGAGTGAG-3' (tink-F) and 5'-CATCTGCCCACTTCCTATTTG-3' (tink-R).

The DNA sequence of *emc^tink^* allele was determined by Sanger sequencing using BigDye Terminator V3.1 Cycle Sequencing Kits (Thermo Fisher Scientific), according to the manufacturers' instruction.
The same primers that amplify *emc* locus (tink-F and tink-R) were used to perform cycle sequencing.

## *In situ* hybridization

Antisense and sense *emc* probes labeled with Digoxigenin were generated from a cDNA clone of *emc*, LD10532 (obtained from BDGP), using the DIG RNA labeling mix (Roche) , according to the manufacturers' instructions. Hybridizations were done at 59 °C for 15 hr in a buffer containing 50 % formamide, 5x SSC (750 mM NaCl, 75 mM sodium citrate) (pH 5.0), 50 μg/mL Heparin, 100 μg/mL denatured salmon sperm DNA, and 0.1 % Tween-20. Washes, treatment with mouse anti-DIG antibody (Roche), and histochemical staining were done as described previously [@Kosman1991; @Tautz1989].
Images were obtained with the differential interference microscope (Axioskop 2 plus, Carl Zeiss) and processed by WinROOF 2015 (Mitani Corporation).

## Quantitative PCR

To quantify the gene products of *da*, *ex*, *Myo31DF*, and *Myosin61F* (*Myo61F*) in wild-type and *emc* mutant embryos, I performed quantitative PCR as previously described [@Okumura2015].
Total RNA was isolated from 30 embryos (stage 12-13) of wild-type or *emc^AP6^* homozygous flies using Isogen (Nippon Gene, Tokyo), and 1 μg of each total RNA sample was used for cDNA synthesis, which was performed using the PrimeScript RT reagent Kit with gDNA Eraser (Takara), according to the product manual.
The amount of *da*, *Myo31DF*, or *Myo61F* transcript was normalized to that of a housekeeping gene, *Actin5C* (*Act5C*).

The primers used to amplify *da* were \seqsplit{5'-TGATGCGTTGACCTCGTTTGTGC-3'} and \seqsplit{5'-AAGTGCTGCATTGGACGTGTTGC-3'}.
The primers used to amplify *Myo31DF* were \seqsplit{5'-ATTTGGCGCACAATTGCCGCTG-3'} and \seqsplit{5'-TTGGCCGTGGATTTCAAGTGCTG-3'}.
The primers used to amplify *ex* were \seqsplit{5'- AGCTACACTTTCGCGTGCAGTTC-3'} and \seqsplit{5'- TGCAAGATGTTGTGGCGCAGC-3'}.
The primers used to amplify *Myo31DF* were \seqsplit{5'-ATTTGGCGCACAATTGCCGCTG-3'} and \seqsplit{5'-TTGGCCGTGGATTTCAAGTGCTG-3'}.
The primers used to amplify *Myo61F* were \seqsplit{5'- AATCGTGGCCAGCATTCTGCATC-3'} and \seqsplit{5'- TGCGTTTACACCCAGCAAACGAG-3'}.
The primers used to amplify *Act5C* were \seqsplit{5'-TTCGCTGTCCACCTTCCAGCAG-3'} and \seqsplit{5'-AAGCACTTGCGGTGCACAATGG-3'}.
Quantitative PCR was performed using SYBR Premix Ex Taq II (Takara) and the Real Time PCR System (7300, Applied Biosystems), according to the manufacturers' instructions.
The expression level of the genes was measured as biological triplicates of technical duplicates.
The mean amounts of transcript were compared by Welch's t-test, and a P-value (p) \< 0.05 was considered significant.

## Western blot analysis

Each protein sample was prepared from fifty wild-type or *emc^AP6^* embryos.
The embryos were homogenized in RIPA buffer (150 mM NaCl, 50 mM Tris-HCl (pH 8.0), 1 % of Triton-X-100) with cOmplete EDTA-free Protease Inhibitor (Sigma-Aldrich).
After centrifugation for 5 minutes at 8,000 x g, the supernatant was incubated for 3 minutes at 95 °C, and the protein extract was applied to a 7% SDS-PAGE gel.
The separated proteins were transferred onto an Immobilon-P membrane (Millipore) according to the manufacturer's instructions.
The primary antibodies used were rat anti-*D*E-Cadherin (1:100, DSHB) and rat anti-Tubulin alpha YL1/2 (1: 1000, AbD Serotec).
To detect the proteins, HRP-conjugated secondary antibodies (1:1000, Jackson ImmunoResearch) and Clarity Western ECL Substrate (Bio-Rad) were used.

## Quantification of anti-*D*E-Cad antibody staining

Embryos were stained with anti-*D*E-Cad and anti-PY20 antibodies.
Optical sections of the hindgut epithelium were captured by a confocal microscope (LSM880, Carl Zeiss), and converted to a z-projection image using ImageJ Fiji.
Ten boundaries were randomly selected, and the mean signal intensity along the boundaries was calculated.
For normalization, the ratio of the intensity of the anti-*D*E-Cad to PY20 staining was calculated using ImageJ Fiji [@Schindelin2012].

## Generation of mosaic tissue in the epithelium of the embryonic hindgut

Mosaic embryos composed of *emc^AP6^* homozygous cells and rescued cells (*emc^AP6^* homozygous cells overexpressing *UAS-emc::GFP*) were generated using the *Aloxg-Gal4* system as previously described [@Nakazawa2012]. *Cre* was expressed maternally and zygotically by *P{y^+mDint2^=Crey}*.
Mosaic embryos were selected based on the *GFP* expression and the absence of anti-β-galactosidase antibody staining.

## Quantification of anti-Myo31DF antibody staining

*emc^AP6^* mutant embryos with *UAS-emc::GFP* overexpressing clonal cells were generated as described above.
These embryos were stained with anti-Myo31DF antibody.
Optical sections of the hindgut epithelium were captured by a confocal microscope (LSM880, Carl Zeiss), and converted to a z-projection image using ImageJ Fiji.
At least 7 *emc* mutant cells and wild-type cells (*emc* mutant cells overexpressing *emc::GFP*) were selected, and the mean signal intensity of Myo31DF along the apical side was measured using ImageJ Fiji [@Schindelin2012].

## Analysis of cell chirality and apical size in hindgut epithelial cells

Cell chirality was analyzed as previously described [@Taniguchi2011].
Briefly, the images of apical cell boundaries at the focal plane corresponding to the adherence junctions, which were detected by anti-*D*E-Cad antibody staining, in the dorsal part of the embryonic hindgut just before its rotation (at stage 12) were obtained using a confocal microscope (LSM880, Carl Zeiss).
Based on these images, the angles ($\theta$) between the anterior-posterior axis of the hindgut tube and each cell boundary were determined by ImageJ Fiji [@Schindelin2012].
To quantify the LR asymmetric slanting of the apical cell boundaries in the hindgut epithelial cells, a chirality index was calculated as $(N_{R} - N_{L})/(N_{R} + N_{L})$ , where $N_{R}$ was the number of boundaries with 0° < $\theta$ < 90°, and $N_{L}$ was the number of boundaries with -90° < $\theta$ < 0°, for each embryo.

The area of the apical surface of the hindgut epithelial cells was measured using the same confocal images with ImageJ Fiji [@Schindelin2012].

The chirality index and apical area were measured in a double-blind manner, in which the person analyzing the cell boundaries did not know the genotypes of the embryos.
The mean values of the chirality indexes and apical sizes were compared by Tukey-Kramer's multiple test, and p < 0.05 was considered significant.

## Analysis of the live cell chirality

The *emc^AP6^* homozygous embryos overexpressing *UAS-myristoylated GFP* (*UAS-myrGFP*) driven by *byn-Gal4* were collected and dechorionated in 50% haiter (Kao).
After washing the embryos, they were mounted with Halocarbon oil 27 (Sigma-Aldrich).
The images of the apical cell boundaries were obtained using a confocal microscope (LSM880, Carl Zeiss).
After the imaging, the embryos were cultured at 25 °C for 3 hr, and the rotation direction of the hindgut was evaluated.
Chirality index of the living hindgut was computed as described above.

## Counting the number of nuclei in the hindgut and the calculation of the hindgut perimeter

The embryos were stained with anti-Lamin C antibody [@Riemer1995].
The images of the whole hindgut shape of wild-type and *emc^tink^* homozygous embryos were obtained using a confocal microscope (LSM880, Carl Zeiss).
The centroids of the Lamin C positive nuclei in the hindgut were manually selected.
The number of the nuclei and the 3D positions of the nuclear centroid were obtained with ImageJ Fiji [@Schindelin2012].

\clearpage

\mbox{}\newpage

