E and ID proteins regulate cell chirality and left-right asymmetric
development in *Drosophila*

**by**

**Tomoki Ishibashi**

***DOCTORAL THESIS***

Presented to the Department of the Biological Science, Osaka University

for the Degree of Doctor of Philosophy (Science)

**Supervisor: Dr. Kenji Matsuno**

Department of Biological Science, Osaka University

1-1 Machikaneyama-cho, Toyonaka, Osaka 560-0043, Japan

**Committee Member 1: Dr. Hiroki Nishida**

Department of Biological Science, Osaka University

1-1 Machikaneyama-cho, Toyonaka, Osaka 560-0043, Japan

**Committee Member 2: Dr. Hiroshi Sasaki**

Graduate School of Frontier Biosciences, Osaka University

1-3 Yamadaoka, Suita, Osaka, 565-0871, Japan

**Author: Tomoki Ishibashi**

Department of Biological Science, Osaka University

1-1 Machikaneyama-cho, Toyonaka, Osaka 560-0043, Japan

Chapter 1. Summary\
===================

How left-right (LR) asymmetry forms in the animal body is a fundamental
problem in Developmental Biology. While the mechanisms for LR asymmetry
are well studied in some species, they are still poorly understood in
invertebrates. Our group previously showed that the intrinsic LR
asymmetry of cells (designated as cell chirality) drives LR asymmetric
development in the *Drosophila* embryonic hindgut, although the
machinery of the cell chirality formation remains elusive. Here, I found
that the *Drosophila* homolog of the *Id* gene, *extra macrochaetae*
(*emc*), is required for the normal LR asymmetric morphogenesis of this
organ. Id proteins, including Emc, are known to interact with and
inhibit E-box-binding proteins (E proteins), such as *Drosophila*
Daughterless (Da). The suppression of *da* by wild-type *emc* was
essential for cell chirality formation and for normal LR asymmetric
development of the embryonic hindgut. *Myosin31DF* (*Myo31DF*), which
encodes the *Drosophila* Myosin ID protein, is known to regulate cell
chirality. I further showed that Emc-Da regulates cell chirality
formation, in which Emc functions upstream of or parallel to Myo31DF.
Abnormal Id-E protein regulation is involved in various human diseases.
My results suggest that defects in cell shape may contribute to the
pathogenesis of such diseases.

Chapter 2. Introduction\
========================

Left-right asymmetry in animals
-------------------------------

The Bilateria have three body axes: anterior-posterior (AP),
dorsal-ventral (DV), and left-right (LR). The mechanisms by which these
three body axes are formed has been a central focus in the field of
Developmental Biology (Arendt & Nübler-Jung, 1997; Kimelman & Martin,
2012; Levin, 2005). The mechanisms for AP and DV axis formation are well
understood in various bilaterian species (Arendt & Nübler-Jung, 1997;
Kimelman & Martin, 2012; Riechmann & Ephrussi, 2001). The molecular
mechanisms of the LR axis formation have also been well studied,
although mostly in vertebrates (Blum & Ott, 2018; Coutelis et al., 2014;
Kimelman & Martin, 2012; Riechmann & Ephrussi, 2001). For example,
*Nodal*, *Lefty1/2*, and *Pitx2* were identified as the core components
of the pathway required for LR axis formation in vertebrates (Blum &
Ott, 2018; Coutelis et al., 2014; Hamada et al., 2002; Hirokawa et al.,
2006; Levin, 2005; T. Nakamura & Hamada, 2012; Okumura et al., 2008;
Tabin, 2006). In some vertebrates, the LR symmetry is first broken by an
LR directional flow of extra-embryonic fluid, which is induced by
ciliary rotation in the node or its equivalent tissue in early embryos
(Hirokawa et al., 2006; McGrath et al., 2003; T. Nakamura & Hamada,
2012; Tabin & Vogan, 2003; Yoshiba & Hamada, 2014). In contrast, in
invertebrates the mechanisms of LR-asymmetry formation remain largely
unclear, although a few excellent studies have unveiled basic concepts
behind the directional LR asymmetry formation in nematodes, snails, and
*Drosophila* (Asami et al., 2008; Bergmann, 2003; Blum & Ott, 2018;
Coutelis et al., 2014; R. Kuroda et al., 2009; Naganathan et al., 2014;
Okumura et al., 2008; Pohl & Bao, 2010; Shibazaki et al., 2004). Some
invertebrate species develop LR asymmetric body structures using
mechanisms arising from the intrinsic chirality of blastomeres or cells
in tissues, which is distinct from the mechanism used in vertebrates,
indicating that the processes for directional LR symmetric development
diverged in evolution (Blum & Ott, 2018; Burdine & Schier, 2000;
Coutelis et al., 2014; T. Nakamura & Hamada, 2012; Okumura et al.,
2008). Nevertheless, our group have been studying the mechanisms of LR
asymmetry formation in *Drosophila melanogaster*, with the hope of
revealing broadly applicable concepts for LR asymmetry formation across
the phyla (Blum & Ott, 2018; Coutelis et al., 2014; Inaki et al., 2016;
Juan et al., 2018; Tingler et al., 2018).

Several organs in *Drosophila* show a directional LR asymmetric
morphology (Adam, G., Perrimon, N., Noselli, 2003; Coutelis et al.,
2013; T. Hayashi & Murakami, 2001; Hozumi et al., 2006; Ligoxygakis et
al., 2001; Pascual et al., 2004; Petzoldt et al., 2012; Sato et al.,
2015; Spéder et al., 2006). Among these organs, the embryonic gut is the
first to exhibit an LR asymmetric shape during development
(Campos-Ortega, 2015; T. Hayashi & Murakami, 2001; Lengyel & Iwaki,
2002; Ligoxygakis et al., 2001). The embryonic gut can be separated into
three major portions: the foregut, midgut, and hindgut, each of which
exhibits a stereotypic and genetically determined LR-asymmetric
morphology (Campos-Ortega, 2015). Among them, the embryonic hindgut
shows the simplest morphology, in which a monolayer epithelial tube
bends like a hook at its most anterior part (Fig. 1). At an early stage
of embryonic development (stage 12), the hindgut is LR symmetric and
bends toward the ventral side of the embryo (Fig. 1A, A’). At the next
stage of development (stage 12-13), the hindgut rotates anticlockwise
90°, which causes the hindgut to curve rightward and to be LR asymmetric
(Fig. 1B, B’) (T. Hayashi & Murakami, 2001; Hozumi et al., 2006).
Neither cell division nor apoptosis is involved in this rotation
(Campos-Ortega, 2015; Lengyel & Iwaki, 2002; Wells et al., 2013).
Moreover, the embryonic hindgut epithelial tube, but not the surrounding
visceral muscles, is sufficient for this rotation (Hozumi et al., 2006;
M. Nakamura et al., 2013). Thus, LR asymmetric cell deformation of the
hindgut epithelial cells themselves may contribute to the hindgut
rotation.

Cell chirality
--------------

In agreement with this idea, our group previously reported that the
epithelial cells of the embryonic hindgut exhibit an LR-asymmetric shape
in their apical surface before hindgut rotation (Hatori et al., 2014;
Inaki et al., 2016, 2018b; Taniguchi et al., 2011). Considering that the
hindgut epithelial cells also have apical-basal polarity, the
three-dimensional structure of these asymmetric hindgut epithelial cells
cannot be superimposed onto their mirror image; they therefore fulfill
the definition of chirality (Taniguchi et al., 2011). Thus, our group
designated this LR-asymmetric property of the hindgut epithelial cells
as “planar cell chirality” (Taniguchi et al., 2011). After hindgut
rotation, the planar cell chirality dissolves, and the epithelial cells
become LR symmetric (Taniguchi et al., 2011). *In silico* models
suggested that this dissolution of cell chirality is sufficient to
induce the LR asymmetric rotation of the embryonic hindgut (Inaki et
al., 2018a; Taniguchi et al., 2011). Our group also demonstrated that
the dissolution of cell chirality induces LR-asymmetric cell sliding in
the hindgut epithelium, which involves LR-asymmetric changes in the
relative positions of cells without their junctional remodeling, and
leads to hindgut rotation (Inaki et al., 2018a). Other groups showed
that cell chirality is also involved in the LR-asymmetric morphogenesis
of the *Drosophila* male genital disc and adult gut (Petzoldt et al.,
2012; Sato et al., 2015).

In the *Drosophila* hindgut, cell chirality was found to be an intrinsic
property of each individual cell (Hatori et al., 2014). Consistent with
this finding *in vivo*, cell chirality was recently observed in various
cultured cells of vertebrates and in blastomeres of *Caenorhabditis
elegans* and snails, implying that the cell chirality is a common and
intrinsic feature of cells (Chen et al., 2012; R. Kuroda et al., 2009;
Naganathan et al., 2014; Pohl & Bao, 2010; Tamada & Igarashi, 2017;
Tamada et al., 2010; Tee et al., 2015; Wan et al., 2011; Yamanaka &
Kondo, 2015). Although Formin is reported to be essential for the
formation of cell chirality in mammalian cells and snail blastomeres,
the mechanisms underlying cell-chirality formation remain unclear
(Davison et al., 2016; Tee et al., 2015).

Myosin ID in cell chirality and LR asymmetry
--------------------------------------------

In *Drosophila*, *Myosin31DF* (*Myo31DF*), which encodes a Myosin ID
(MyoID) ortholog, has a critical role in switching the enantiomeric
states of chiral cells (Hatori et al., 2014; Taniguchi et al., 2011). In
*Myo31DF* mutants, the chirality of the hindgut epithelial cells is the
mirror image of their wild-type counterparts, and the direction of the
hindgut rotation becomes reversed (Hatori et al., 2014; Taniguchi et
al., 2011). In addition to the hindgut, *Myo31DF* mutation switches the
cell chirality in the male genital disc and adult gut, and consequently
reverses the LR asymmetry of these organs as well (Sato et al., 2015;
Spéder et al., 2006). *Abdominal-B* (*Abd-B*), a Hox gene, is required
to activate *MyoID* expression in the embryonic hindgut and the male
genital disc (Coutelis et al., 2013). In addition, Myo31DF was shown to
interact with the adherens junctions and Dachsous (Ds) to regulate the
LR asymmetry of the genital disc and adult gut, respectively
(González-Morales et al., 2015). These results suggest that Myo31DF
functionally and physically interacts with various cellular factors to
execute its roles in LR asymmetric development. However, the molecular
functions of Myo31DF in these cases are still elusive. Notably, MyoID is
also reported to have essential roles in LR asymmetric development in
vertebrates, suggesting an evolutionary conservation of this overall
purpose, although the molecular functions of MyoID may vary (Tingler et
al., 2018).

E protein and Id proteins
-------------------------

Given that Myo31DF is known to interact functionally and/or physically
with various cellular components, including a Hox transcription factor,
other classes of transcription factors that globally contribute to
embryonic development may also be important for its functions. The basic
helix-loop-helix (bHLH) protein family is one such class of
transcription factors (Massari & Murre, 2000; Wang & Baker, 2015a). The
bHLH proteins are broadly conserved in various organisms and regulate
the transcription of a large number of target genes (Gyoja, 2017;
Massari & Murre, 2000; Morgenstern & Atchley, 1999; Wang & Baker,
2015a). One class of bHLH proteins, called E proteins, bind to the
E-boxes in the *cis*-acting regulatory region of target genes as a homo-
or heterodimer, and promote gene transcription (Massari et al., 1999;
Murre et al., 1989). The inhibitor of DNA-binding/differentiation
proteins, also known as Id proteins, form a heterodimer with bHLH
proteins, including the E proteins, to inhibit their DNA binding
(Benezra et al., 1990; Ling et al., 2014). Therefore, Id proteins act as
antagonists of the E proteins (Benezra et al., 1990; Ling et al., 2014),
and the balance between E and Id proteins controls various cellular
events through the transcriptional regulation of E-protein target genes
(Bhattacharya & Baker, 2011; Ling et al., 2014; Wang & Baker, 2015a).
For example, the balance between E and Id proteins is reported to be
important for cell-cycle regulation (Wang & Baker, 2015a). In mammals, E
proteins induce the expression of genes encoding CDK inhibitors (CDKIs),
which inhibit cell-cycle progression (Pagliuca et al., 2000; Prabhu et
al., 1997; Sloan et al., 1996). Id proteins promote cell-cycle
progression, because they prevent the E protein-dependent expression of
genes encoding CDKIs (Lasorella et al., 2014). The E and Id proteins
also have important regulatory roles in cell differentiation (Belle &
Zhuang, 2014; Murre, 2005; Wang & Baker, 2015a). In general, E proteins
promote cell differentiation (Massari & Murre, 2000; Wang & Baker,
2015a). In undifferentiated cells, the level of Id proteins is
increased, which inhibits the E-box transcriptional activity of E
proteins and consequently maintains the multipotency of the cells
(Lasorella et al., 2014; Nam & Benezra, 2009; Niola et al., 2012).
Therefore, for example, mutants of the Id protein-encoding gene in
*Drosophila* show ectopic neural differentiation (Ellis et al., 1990;
Garrell & Modolell, 1990).

*Drosophila* has single orthologous genes encoding the E and Id
proteins, *daughterless* (*da*) and *extra macrochaetae* (*emc*),
respectively (Caudy et al., 1988a, 1988b; Ellis et al., 1990; Garrell &
Modolell, 1990). Therefore, the functions of the E and Id proteins are
relatively easy to study in *Drosophila*, compared with those in
vertebrate systems, which have multiple E and Id proteins (Gyoja, 2017).
In *Drosophila*, Da forms a homodimer or a heterodimer with other bHLH
proteins to upregulate the expression of target genes, whereas Emc
functions as an antagonist of Da by forming an inactive heterodimer with
it (Cabrera & Alonso, 1991; Garrell & Modolell, 1990; Murre et al.,
1989). The transcriptional regulations involving Emc and Da control
various developmental steps in *Drosophila*, including the spatial
organization of neuroblasts, wing development, and follicle cell
differentiation (Bhattacharya & Baker, 2011; Cummings & Cronmiller,
1994; de Celis et al., 1995; Ellis et al., 1990; Garrell & Modolell,
1990; Smith et al., 2002). Emc and Da also modulate various cell
signaling pathways, such as the Notch pathway, in which their roles have
been well studied (Baonza et al., 2000; Cummings & Cronmiller, 1994).

[]{#hippo-pathway .anchor}In this paper, I report for the first time
that the interaction between E and Id proteins controls LR-asymmetric
development. I reveal that a mutation of *emc* resulted in the
hyperactivation of Da, which subsequently led to LR-asymmetry defects of
the embryonic hindgut in *Drosophila*. These LR defects were coupled
with the loss of cell chirality in the hindgut epithelium. Thus, the
suppression of *da* by *emc* was essential for the formation of dextral
cell chirality and the LR-asymmetric development of the hindgut,
although *da* was not required for these events. I also report a close
relationship between Myo31DF and Emc, and that Myo31DF functioned
downstream of or parallel to Emc in LR-asymmetric development and
dextral cell chirality formation. Disorders in the E and Id protein
balance are known to lead to various human diseases, which are mostly
attributed to defects in cell-fate decision or cell-cycle regulation
(Ling et al., 2014; Wang & Baker, 2015a). However, here I showed that an
aberration in the interaction between E and Id proteins can also affect
cell shape. Therefore, my results may introduce a novel avenue for
understanding the pathogenesis of diseases attributed to the
dysregulation of E and Id protein interactions.

Chapter 3. Experimental Procedures\
===================================

*Drosophila* strains and genetic crosses
----------------------------------------

*Canton-S* was used as the wild-type (WT) strain. The following mutants
were used: *emc^tink^*, a null allele induced by ethyl methanesulfonate
in this study; *emc^AP6^*, an amorphic allele (Bloomington \#36544;
Ellis, 1994); *emc^2^*, a hypomorphic allele (Kyoto DGRC \#101588; Ellis
et al., 1990); *da^10^*, an amorphic allele (Bloomington \#5531; Wülbeck
et al., 1994); and *Myo31DF^K2^*, an amorphic allele (Petzoldt et al.,
2012). The following UAS lines were used: *UAS-emc::GFP* (Popova et al.,
2011); *UAS-da* (Bloomington \#51669; Giebel et al., 1997);
*UAS-myrGFP*; and *UAS-Myo31DF::mRFP* (Hatori et al., 2014). The
following *Gal4*-driver lines were used: *NP2432*, a hindgut
epithelium-specific driver (Kyoto DGRC \#104201; S. Hayashi et al.,
2002); *byn-Gal4*, a hindgut epithelium-specific driver; and
*hand-Gal4*, a visceral muscle cell-specific driver (J. Kuroda et al.,
2012). To express *Cre* in *Drosophila*, *y^1^, w^67c23^
P{y^+mDint2^=Crey}1b* was used to generate mosaic tissues (Nakazawa et
al., 2012).

Mutations on the first and second chromosome were balanced with *FM7c,
P{ftz/lacC}YH1* and *CyO, P{en1}wg^en11^*, respectively. Mutations on
the third chromosome were balanced with *TM3, P{ftz-lacZ.ry^+^}TM3,
Sb^1^ ry^\*^, TM6B, P{iab-2(1.7)lacZ}6B, Tb1*, or *TM3,
P{GAL4-twi.G}2.3, P{UAS-2xEGFP}AH2.3, Sb^1^ Ser^1^*. All genetic crosses
were performed at 25 °C on a standard *Drosophila* culture medium.

Preparation of *Drosophila* embryos
-----------------------------------

*Drosophila* embryos were collected and dechorionated in 50% haiter
(Kao). The dechorionated embryos were fixed by 6% of paraformaldehyde in
PBS (130 mM NaCl, 7 mM Na~2~HPO~4~, and 3 mM NaH~2~PO~4~) for 30
minutes. After the fixation, the vitelline membrane of the embryos was
removed by washing in 100% methanol or by the manual dissection. To
preserve the fixed embryos, they were kept in 100 % methanol at -20 ºC.

X-Gal Staining of embryos
-------------------------

To select the homozygous embryos of mutants, X-gal staining was
performed (Sullivan et al., 2000). The embryos expressing *lacZ* were
stained with X-Gal (5-bromo-4-chloro-3-indolyl-β-d-galactopyranoside) in
X-Gal buffer (3 mM K~3~Fe\[CN\]~6~, 3 mM K~4~Fe\[CN\]~6~, 0.9 mM
MgCl~2~, and 0.1 % (w/v) Tween-20 in PBS) at 37 °C for 1 hr.

Immunolabeling of embryos
-------------------------

Fixed embryos were treated with 2 % Block Ace (Yukijirushi) in PBT
[[]{#OLE_LINK4 .anchor}]{#OLE_LINK3 .anchor}(130 mM NaCl, 7 mM
Na~2~HPO~4~, 3 mM NaH~2~PO~4~, and 0.1 % (w/v) Triton-X-100). After
washing the embryos with PBT, they were incubated with the primary
antibodies solution in Can Get Signal Solution B (TOYOBO), IMMUNO SHOT
Strong (Cosmo Bio), or 2% Block Ace (Yukijirushi) in PBT in 2 mL tubes
(Eppendorf) at room temperature for 3 hr or at 4 °C over night. The
embryos were washed with PBT, and incubated with the secondary
antibodies solution at room temperature for 3 hr or at 4 °C over night.
After washing the embryos, they were mounted with 60% glycerol.

The following primary antibodies were used at the dilution indicated:
mouse anti-phosphotyrosine antibody (PY20, the marker for cell membrane)
(1:1000, BD Biosciences), rat anti-DE-Cadherin (Cad) antibody (1:100,
DSHB), rabbit anti-GFP antibody (1:500, MBL), rabbit anti-RFP antibody
(1:1000, MBL), mouse anti-Wingless antibody (1:1000, DSHB), mouse
anti-Engrailed (1:1000, DSHB), mouse anti-Crumbs (1:1000, DSHB), chicken
anti-β-galactosidase antibody (1:500, Abcam), and mouse anti-Lamin C
antibody (1:1000, DSHB).

The following secondary antibodies were used at the dilution indicated:
anti-mouse IgG-Cy3 (1:1000, Jackson ImmunoResearch), anti-mouse
IgG-Alexa Fluor 488 (1:1000, Molecular Probes), anti-mouse IgG-Cy5
(1:1000, Jackson ImmunoResearch), anti-rabbit IgG-Alexa Fluor 488
(1:1000, Molecular Probes), anti-rabbit IgG-Cy3 (1:1000, Jackson
ImmunoResearch), anti-rabbit IgG-Cy5 (1:1000, Jackson ImmunoResearch),
anti-rat IgG-Cy3 (1:1000, Jackson ImmunoResearch), anti-rat IgG-Alexa488
(1:1000, Jackson ImmunoResearch), anti-rat IgG-Cy5 (1:1000, Jackson
ImmunoResearch), and anti-chicken IgY-Alexa488 (1:1000, Jackson
ImmunoResearch). To detect F-actin, rhodamine phalloidin was used
(1:250, Thermo Fisher Scientific).

To evaluate LR defects of the hindgut, embryos were mounted in 50%
glycerol and observed under a differential interference microscope
(Axioskop 2 plus, Zeiss). To obtain images of the hindgut, embryos were
treated with a tissue-clearing reagent (methyl salicylate), and examined
by confocal microscopy (LSM880, Carl Zeiss). The images were processed
using Fiji software, Adobe Photoshop CS3 (Adobe Systems), or Adobe
Illustrator CS3 (Adobe Systems).

DNA amplification by PCR and sequencing
---------------------------------------

Genomic DNA (gDNA) were collected from 30 embryos of wild-type or
*emc^tink^* homozygous embryos using the standard protocol described
previously (Sullivan et al., 2000). To amplify the *emc* locus, I
performed polymerase chain reaction (PCR) using ExTaq (Takara),
according to the product manual. The primers used to amplify *emc* were
5'-TCGAACATCGACAGAGTGAG-3' (tink-F) and 5'-CATCTGCCCACTTCCTATTTG-3'
(tink-R).

The DNA sequence of *emc^tink^* allele was determined by Sanger
sequencing using BigDye Terminator V3.1 Cycle Sequencing Kits (Thermo
Fisher Scientific), according to the manufacturers’ instruction. The
same primers that amplify *emc* locus (tink-F and tink-R) were used to
perform cycle sequencing.

*In situ* hybridization
-----------------------

Antisense and sense *emc* probes labeled with Digoxigenin were generated
from a cDNA clone of *emc*, LD10532 (obtained from BDGP), using the DIG
RNA labeling mix (Roche) , according to the manufacturers’ instructions.
Hybridizations were done at 59 °C for 15 hr in a buffer containing 50 %
formamide, 5x SSC (750 mM NaCl, 75 mM sodium citrate) (pH 5.0), 50 μg/mL
Heparin, 100 μg/mL denatured salmon sperm DNA, and 0.1 % Tween-20.
Washes, treatment with mouse anti-DIG antibody (Roche), and
histochemical staining were done as described previously (Kosman et al.,
1991; Tautz & Pfeifle, 1989). Images were obtained with the differential
interference microscope (Axioskop 2 plus, Carl Zeiss) and processed by
WinROOF 2015 (Mitani Corporation).

Quantitative PCR
----------------

To quantify the gene products of *da*, *ex*, *Myo31DF*, and *Myosin61F*
(*Myo61F*) in wild-type and *emc* mutant embryos, I performed
quantitative PCR as previously described (Okumura et al., 2015). Total
RNA was isolated from 30 embryos (stage 12-13) of wild-type or
*emc^AP6^* homozygous flies using Isogen (Nippon Gene, Tokyo), and 1 μg
of each total RNA sample was used for cDNA synthesis, which was
performed using the PrimeScript RT reagent Kit with gDNA Eraser
(Takara), according to the product manual. The amount of *da*,
*Myo31DF*, or *Myo61F* transcript was normalized to that of a
housekeeping gene, *Actin5C* (*Act5C*). The primers used to amplify *da*
were 5'-TGATGCGTTGACCTCGTTTGTGC-3' and 5'-AAGTGCTGCATTGGACGTGTTGC-3'.
The primers used to amplify *Myo31DF* were 5'-ATTTGGCGCACAATTGCCGCTG-3'
and 5'-TTGGCCGTGGATTTCAAGTGCTG-3'. The primers used to amplify *ex* were
5'- AGCTACACTTTCGCGTGCAGTTC-3' and 5'- TGCAAGATGTTGTGGCGCAGC-3'. The
primers used to amplify *Myo31DF* were 5'-ATTTGGCGCACAATTGCCGCTG-3' and
5'-TTGGCCGTGGATTTCAAGTGCTG-3'. The primers used to amplify *Myo61F* were
5'- AATCGTGGCCAGCATTCTGCATC-3' and 5'- TGCGTTTACACCCAGCAAACGAG-3'. The
primers used to amplify *Act5C* were 5'-TTCGCTGTCCACCTTCCAGCAG-3' and
5'-AAGCACTTGCGGTGCACAATGG-3'. Quantitative PCR was performed using SYBR
Premix Ex Taq II (Takara) and the Real Time PCR System (7300, Applied
Biosystems), according to the manufacturers’ instructions. The
expression level of the genes was measured as biological triplicates of
technical duplicates. The mean amounts of transcript were compared by
Welch’s t-test, and a P-value (p) &lt; 0.05 was considered significant.

Western blot analysis
---------------------

Each protein sample was prepared from fifty wild-type or *emc^AP6^*
embryos. The embryos were homogenized in RIPA buffer (150 mM NaCl, 50 mM
Tris-HCl (pH 8.0), 1 % of Triton-X-100) with cOmplete EDTA-free Protease
Inhibitor (Sigma-Aldrich). After centrifugation for 5 minutes at 8,000 x
g, the supernatant was incubated for 3 minutes at 95 °C, and the protein
extract was applied to a 7% SDS-PAGE gel. The separated proteins were
transferred onto an Immobilon-P membrane (Millipore) according to the
manufacturer’s instructions. The primary antibodies used were rat
anti-DE-Cadherin (1:100, DSHB) and rat anti-Tubulin alpha YL1/2 (1:
1000, AbD Serotec). To detect the proteins, HRP-conjugated secondary
antibodies (1:1000, Jackson ImmunoResearch) and Clarity Western ECL
Substrate (Bio-Rad) were used.

Quantification of anti-DE-Cad antibody staining
-----------------------------------------------

Embryos were stained with anti-DE-Cad and anti-PY20 antibodies. Optical
sections of the hindgut epithelium were captured by a confocal
microscope (LSM880, Carl Zeiss), and converted to a z-projection image
using ImageJ Fiji. Ten boundaries were randomly selected, and the mean
signal intensity along the boundaries was calculated. For normalization,
the ratio of the intensity of the anti-DE-Cad to PY20 staining was
calculated using ImageJ Fiji (Schindelin et al., 2012).

Generation of mosaic tissue in the epithelium of the embryonic hindgut
----------------------------------------------------------------------

Mosaic embryos composed of *emc^AP6^* homozygous cells and rescued cells
(*emc^AP6^* homozygous cells overexpressing *UAS-emc::GFP*) were
generated using the *Aloxg-Gal4* system as previously described
(Nakazawa et al., 2012). *Cre* was expressed maternally and zygotically
by *P{y^+mDint2^=Crey}*. Mosaic embryos were selected based on the *GFP*
expression and the absence of anti-β-galactosidase antibody staining.

Quantification of anti-Myo31DF antibody staining
------------------------------------------------

*emc^AP6^* mutant embryos with *UAS-emc::GFP* overexpressing clonal
cells were generated as described above. These embryos were stained with
anti-Myo31DF antibody. Optical sections of the hindgut epithelium were
captured by a confocal microscope (LSM880, Carl Zeiss), and converted to
a z-projection image using ImageJ Fiji. At least 7 *emc* mutant cells
and wild-type cells (*emc* mutant cells overexpressing *emc::GFP*) were
selected, and the mean signal intensity of Myo31DF along the apical side
was measured using ImageJ Fiji (Schindelin et al., 2012).

Analysis of cell chirality and apical size in hindgut epithelial cells
----------------------------------------------------------------------

Cell chirality was analyzed as previously described (Taniguchi et al.,
2011). Briefly, the images of apical cell boundaries at the focal plane
corresponding to the adherence junctions, which were detected by
anti-DE-Cad antibody staining, in the dorsal part of the embryonic
hindgut just before its rotation (at stage 12) were obtained using a
confocal microscope (LSM880, Carl Zeiss). Based on these images, the
angles (*θ*) between the anterior-posterior axis of the hindgut tube and
each cell boundary were determined by ImageJ Fiji (Schindelin et al.,
2012). To quantify the LR asymmetric slanting of the apical cell
boundaries in the hindgut epithelial cells, a chirality index was
calculated as (*N~R~* – *N~L~*)/(*N~R~* + *N~L~*) , where *N~R~* was the
number of boundaries with 0° &lt; *θ* &lt; 90°, and *N~L~* was the
number of boundaries with -90° &lt; *θ* &lt; 0°, for each embryo.

The area of the apical surface of the hindgut epithelial cells was
measured using the same confocal images with ImageJ Fiji (Schindelin et
al., 2012).

The chirality index and apical area were measured in a double-blind
manner, in which the person analyzing the cell boundaries did not know
the genotypes of the embryos. The mean values of the chirality indexes
and apical sizes were compared by Tukey-Kramer’s multiple test, and p
&lt; 0.05 was considered significant.

Analysis of the live cell chirality
-----------------------------------

The *emc^AP6^* homozygous embryos overexpressing *UAS-myristoylated GFP*
(*UAS-myrGFP*) driven by *byn-Gal4* were collected and dechorionated in
50% haiter (Kao). After washing the embryos, they were mounted with
Halocarbon oil 27 (Sigma-Aldrich). The images of the apical cell
boundaries were obtained using a confocal microscope (LSM880, Carl
Zeiss). After the imaging, the embryos were cultured at 25 ºC for 3 hr,
and the rotation direction of the hindgut was evaluated. Chirality index
of the living hindgut was computed as described above.

Counting the number of nuclei in the hindgut and the calculation of the hindgut perimeter
-----------------------------------------------------------------------------------------

The embryos were stained with anti-Lamin C antibody (Riemer et al.,
1995). The images of the whole hindgut shape of wild-type and
*emc^tink^* homozygous embryos were obtained using a confocal microscope
(LSM880, Carl Zeiss). The centroids of the Lamin C positive nuclei in
the hindgut were manually selected. The number of the nuclei and the 3D
positions of the nuclear centroid were obtained with ImageJ Fiji
(Schindelin et al., 2012).

Chapter 4. Result 1: *emc* is involved in LR-asymmetric development of the embryonic hindgut *via* *da*\
========================================================================================================

*emc* is required for LR-asymmetric development of the embryonic hindgut
------------------------------------------------------------------------

In wild-type embryos of *Drosophila* *melanogaster*, the hindgut shows a
left-right (LR) symmetric morphology until embryonic stage 12, although
it curves toward the ventral side of the embryo (Fig. 1A, A’)
(Campos-Ortega, 2015; T. Hayashi & Murakami, 2001). At stage 13, the
hindgut rotates anticlockwise 90°, which makes it LR asymmetric, as it
curves to the right (Fig. 1B, B’). The direction and degree of the
hindgut rotation are stereotypic in wild-type embryos, which rarely
exhibit incomplete rotation, LR inverse, or non-lateral hindgut
phenotypes (Fig. 2) (T. Hayashi & Murakami, 2001; Hozumi et al., 2006).
Therefore, the LR asymmetric morphology of the hindgut is strictly
controlled genetically.

To elucidate the genetic mechanisms of LR asymmetric development in
*Drosophila*, Our group performed a genetic screen to find mutants
showing LR asymmetry defects of the embryonic gut (Hozumi et al., 2006;
J. Kuroda et al., 2012; R. Maeda et al., 2007; M. Nakamura et al., 2013;
Okumura et al., 2015; Taniguchi et al., 2007, 2011). From this screen, a
novel allele of *emc*, *emc^tink^* was identified. Tink was named after
a character who has LR inverse internal organs in a Japanese animated
series, Black Jack by Osamu Tezuka (Tezuka Productions Company,
Limited). The hindgut of *emc^tink^* homozygous embryos showed
LR-asymmetry defects, including LR inversion and LR symmetry (no
laterality), at a frequency of about 25% (Fig. 2 and Fig. 3A). The
contribution of *emc* to the LR asymmetric development of the hindgut
was confirmed by the observation that embryos homozygous for *emc^2^*, a
preexisting allele of *emc*, also showed the LR defects of the hindgut
(Fig. 2). In addition, the hindgut of embryos homozygous for *emc^AP6^*,
a preexisting null allele of *emc*, and those trans-heterozygous for
*emc^tink^* and *emc^AP6^* showed similar LR defects to those of the
*emc^tink^* homozygote (Fig. 2 and Fig. 3B, C). These results suggested
that *emc^tink^* is a null mutant of *emc* genetically. On the other
hand, heterozygotes for *emc^AP6^* did not show LR defects,
demonstrating that these LR inversion phenotypes were recessive (Fig.
2). The size of the PCR fragment of *emc* locus amplified from the
genomic DNA of *emc^tink^* mutant was smaller than that from wild-type
genomic DNA (Fig. 4A). This data suggests that *emc^tink^* mutants lack
partial sequences of the wild-type *emc* locus. To confirm the
deficiency of the *emc* locus in *emc^tink^* mutants, I performed Sanger
sequencing of *emc* gene. *emc^tink^* has a deletion in the coding
sequence of the *emc* locus that results in the loss of a C-terminal
glutamine-rich region in the deduced Emc protein (Fig. 4 B, C). Similar
Gln-rich repeats are found in many *Drosophila* proteins and are known
to have various critical roles (Table 1; Grabowski et al., 1991; Rice &
Liu, 2015; Wharton et al., 1985), including potentially in the Emc
protein (Garrell & Modolell, 1990). My finding that *emc^tink^* was a
null mutant supported the idea that the Gln-rich repeat has an essential
role in Emc’s function.

To investigate the role of *emc* in LR asymmetric development of the
hindgut, I first examined the tissue(s) in which *emc* is required to
generate the normal LR-asymmetric morphology of this organ. First, I
examined the *emc* expression in the hindgut of wild-type embryos by *in
situ* hybridization during development (Fig. 5). *emc* was expressed in
the hindgut primordium at stage 11 in all of the embryos examined (n =
12) (Fig. 5A and A’, Table 2). The *emc* expression continued in the
hindgut at stage 12, when the hindgut started and completed its
rotation, in all cases examined (n = 17) (Fig. 5B and B’, Table 2).
However, *emc* was only weakly expressed in the hindgut at stage 14 in
most of the embryos (14 out of 21), when the hindgut rotation was
completed (Fig. 5C, Table 2). In contrast, a sense probe (negative
control) did not exhibit a strong signal from stage 11-14 in wild-type
embryos (Fig. 5D, D’, E, E’, F, and F’). Thus, *emc* was expressed just
before and during the rotation of the embryonic hindgut. Based on this
expression pattern, I decided to examine the potential role of *emc* in
the tissues of the hindgut.

Our group previously showed that the epithelial cells of the hindgut,
but not of the visceral muscles overlying them, are responsible for the
LR asymmetric development of the hindgut (Hozumi et al., 2006, 2008; M.
Nakamura et al., 2013). Thus, I misexpressed *UAS-emc::GFP* in the
hindgut epithelium of the *emc^AP6^* homozygote using a hindgut
epithelium-specific *Gal4* driver line, *NP2432*. The LR defects
associated with *emc^AP6^* were effectively suppressed in these embryos
(*NP2432&gt;emc::GFP* in *emc^AP6^* in Fig. 2 and Fig. 3D).
*UAS-emc::GFP* is known to have the wild-type functions of *emc* and to
produce the predicted protein size on a Western blot (Popova et al.,
2011). *UAS-emc::GFP* heterozygous embryos did not show the LR defects
(Fig. 2). In *emc^AP6^* homozygotes carrying either *UAS-emc::GFP* or
*NP2432* (negative controls), the LR defects were not suppressed (Fig.
2). The misexpression of *UAS-GFP* driven by *NP2432* also did not
suppress the LR defects of *emc^AP6^* (*NP2432*&gt;*GFP* in *emc^AP6^*
in Fig. 2). In addition, neither *UAS-emc::GFP* expression in the
visceral muscles of the hindgut, driven by *hand-Gal4*, nor the
*emc^AP6^* homozygote carrying only *hand-Gal4* (negative control)
suppressed the LR defects of the *emc^AP6^* homozygote
(*hand&gt;emc::GFP* in *emc^AP6^* and *hand-Gal4*; *emc^AP6\ ^* in Fig.
2 and Fig. 3E) (J. Kuroda et al., 2012). Collectively, these results
suggested that a function of *emc* in the embryonic hindgut epithelium
is required for normal LR asymmetric development of this organ. I also
found that the misexpression of *UAS-emc::GFP* driven by *NP2432* in
wild-type embryos did not lead to LR defects of the embryonic hindgut
(Fig. 2), indicating that excess *emc* activity did not affect the LR
asymmetric development of the hindgut, even though *emc* was required
for this process.

LR-defects of the *emc* mutant hindgut are not caused by abnormal tissue specification
--------------------------------------------------------------------------------------

As described above, *emc* is responsible for the normal LR-asymmetric
development of the embryonic hindgut, where the endogenous expression of
*emc* was detected. Because it was known that Emc is involved in the
differentiation of various tissues, I suspected that the defective
specification of the hindgut tissues may cause the LR-defects of its
asymmetric development. The embryonic hindgut is subdivided into four
domains, the small intestine, the large intestine, the rectum, and the
anal pad (Hoch & Pankratz, 1996; Iwaki et al., 2001; Takashima &
Murakami, 2001) (Fig. 6A). The large intestine is further subdivided
into three domains, the dorsal and ventral domains and boundary cells
(Fuß & Hoch, 2002; Hoch & Pankratz, 1996; Iwaki et al., 2001; Takashima
& Murakami, 2001) (Fig. 6A). Various marker genes are available to
distinguish these domains in the hindgut. Therefore, to determine
whether these domains were normally specified in *emc* mutants, I
examined the expression of these marker genes, *engrailed* (*en*),
*crumbs* (*crb*), *hedgehog* (*hh*), and *wingless* (*wg*), which are
expressed in the dorsal domain of the large intestine, boundary cells,
the rectum, and the anal pad, respectively (Fig. 6A). The expression
patterns detected by the specific antibodies of these genes were similar
between the wild-type and *emc^AP6^* mutant embryos (Fig. 6). These data
suggested that the tissue-specification of the hindgut epithelium was
largely normal in *emc* homozygous embryos, and the role of *emc* in the
cell fate determination of the hindgut tissues was irrelevant to the
formation of the normal LR-asymmetry of the embryonic hindgut.

Suppression of *da* by wild-type *emc* is required for normal LR-asymmetric development of the embryonic hindgut
----------------------------------------------------------------------------------------------------------------

Id proteins, including Emc in *Drosophila*, are known to bind to E
proteins and suppress their activity (Benezra et al., 1990; Ling et al.,
2014). Daughterless (Da) is the E protein in *Drosophila*, and its
activity has been shown to be suppressed by Emc in various tissues and
organs (Cabrera & Alonso, 1991; Garrell & Modolell, 1990; Murre et al.,
1989). Therefore, I speculated that the *da* function might also be
suppressed by wild-type *emc* in the hindgut of wild-type embryos (Fig.
7A). If so, Da should be hyperactivated in the hindgut of *emc*
homozygotes, which might lead to the LR defects of this organ (Fig. 7B).
To examine this possibility, I specifically overexpressed *UAS-da* in
the hindgut epithelium under control of the *NP2432* line
(*NP2432&gt;da* in Fig. 8 and Fig. 9B). These embryos showed LR
asymmetry defects of the hindgut similar to those of the *emc^AP6^*
homozygote, suggesting that the LR defects of the hindgut in *emc*
homozygotes were caused by a derepression of *da* (Fig. 8 and Fig. 9C).
*NP2432* or *UAS-da* alone did not affect the LR asymmetry of the
hindgut (*NP2432-Gal4/+* and *UAS-da/+* in Fig. 3B). Notably, a *da*
null mutant, *da^10^*, also did not show LR defects (Fig. 8 and Fig.
9D), indicating that *da* is not required for the normal LR asymmetric
development of the hindgut, even though its overexpression was
sufficient to impede this process. This result was consistent with my
observation that the misexpression of *emc* in the hindgut epithelium of
wild-type embryos, in which the *da* activity was presumably suppressed
further, did not disrupt the LR asymmetric development of the embryonic
hindgut (Fig. 2). Based on these results, I speculated that a
hyperactivation of *da* due to the absence of wild-type *emc* results in
the LR defects of the hindgut of *emc* mutant embryos.

Given that the hyperactivation of *da* was responsible for the LR
defects in *emc* mutants, I speculated that the LR defects associated
with the *emc* mutation would be suppressed in a double homozygote with
*da*. In agreement with this scenario, embryos double homozygous for
*emc* and *da* did not show the LR defects (Fig. 8 and Fig. 9E). The
expression level of *da* increases in *emc* mutant cells, because the
derepression of Da is known to result in auto-activation of the *da*
gene through a positive-feedback mechanism (Fig. 7B) (Bhattacharya &
Baker, 2011). It was difficult to quantify the induction of *da*
expression in *emc* mutants by *in situ* hybridization, because the *da*
expression was ubiquitous and weak in stage 11-12 embryos. Therefore, I
used quantitative PCR (qPCR) (VanGuilder et al., 2008), which showed
that the *da* transcript level in whole embryos homozygous for
*emc^AP6^* was 15 times the wild-type level (Fig. 10). Taking these
results together, I concluded that Emc is required for the normal
LR-asymmetric development of the embryonic hindgut in wild-type
*Drosophila*, which is mediated by the inhibition of Da activity (Fig.
7).

Chapter 5. Result 2: *emc* is required for the cell chirality formation\
========================================================================

The function of Emc-Da may be closely related to that of Myo31DF in the control of hindgut LR asymmetry
-------------------------------------------------------------------------------------------------------

In *Myo31DF* mutant *Drosophila* homozygotes, the LR asymmetry of
various organs, including the embryonic hindgut, is the mirror image of
that in wild type (Hozumi et al., 2006; Sato et al., 2015; Spéder et
al., 2006). Considering that both *emc* and *Myo31DF* affect the
LR-asymmetry of the embryonic hindgut, I investigated the functional
relationship between these two genes with respect to the LR phenotypes
of the hindgut. *Myo31DF^K2^*, a *Myo31DF* null mutant, showed the
inversion of hindgut LR asymmetry at 80% frequency (Fig. 11 and Fig.
12B), while, as mentioned above, the *emc^AP6^* homozygote showed LR
inversion in the hindgut at about 30% frequency (Fig. 11 and Fig. 12C).
*Myo31DF^K2^; emc^AP6^* double mutants showed the *Myo31DF^K2^*
mutant-like phenotype, which was 80% LR inversion (Fig. 11). This
finding suggested that *Myo31DF* is epistatic to *emc* in the control of
LR-asymmetric development of the embryonic hindgut.

To evaluate this idea further, I examined whether the overexpression of
*Myo31DF* rescued the LR defects of the hindgut in *emc* homozygotes.
First, I confirmed that the LR defects of the *Myo31DF^K2^* homozygote
were efficiently rescued by the overexpression of *UAS-Myo31DF::RFP* in
the hindgut epithelium, driven by *NP2432* (*NP2432*&gt;*Myo31DF::RFP*,
*Myo31DF^K2^* in Fig. 11 and Fig. 12 D). *UAS-Myo31DF::RFP* alone
(negative control) did not rescue the LR defects of the *Myo31DF^K2^*
homozygote (*Myo31DF^K2^*; *UAS-Myo31DF::RFP*/ + in Fig. 11). However,
*UAS-Myo31DF::RFP* overexpressed in the hindgut epithelium of the
*emc^AP6^* homozygote, driven by *NP2432* (*NP2432*&gt;*Myo31DF::RFP*,
*emc^AP6^*) efficiently rescued the LR defects of the hindgut (Fig. 11
and Fig. 12E). As described above, the overexpression of *UAS-da* in the
hindgut epithelium, driven by *NP2432*, resulted in LR defects in the
hindgut of wild-type embryos (*NP2432&gt;da*), suggesting that *da* is
hyperactivated in the hindgut epithelium of *emc* mutants, leading to
defects in the LR asymmetric development of this organ (Fig. 8, Fig. 9B,
Fig. 11, and Fig. 12 F). The LR defects associated with *da*
overexpression were markedly suppressed by the co-overexpression of
*UAS-Myo31DF::RFP* (*NP2432&gt;da&gt;Myo31DF::RFP*), but not of
*UAS-GFP* (negative control, *NP2432&gt;da&gt;GFP*) (Fig. 11 and Fig.
12G). These observations were consistent with the idea that *Myo31DF* is
epistatic to *emc* in the control of the LR-asymmetric development of
the embryonic hindgut. Therefore, Myo31DF functions downstream of or
parallel to Emc in the LR-asymmetric development of this organ.

Based on the observed epistasis between *Myo31DF* and *emc* mutants and
the biochemical function of Emc-Da as a transcription factor, I
suspected that *emc* normally induced the *Myo31DF* expression by
inactivating Da. To test this hypothesis, I compared the expression
level of the *Myo31DF* gene between wild-type and *emc* mutant embryos.
However, I did not detect a statistically significant difference in the
amount of *Myo31DF* mRNA between these two conditions (Fig. 13),
suggesting that Emc-Da does not regulate the transcription of the
*Myo31DF* gene. Next, I speculated that the *emc* downstream target gene
is responsible for the proper localization of Myo31DF protein. In this
case, the loss of *emc* gene may induce Myo31DF mislocalization, which
can cause the LR asymmetric defects of the hindgut. To test this
possibility, I generated *UAS-emc::GFP*-overexpressing clonal cells on
the *emc^AP6^* mutant background and stained endogenous Myo31DF with
anti-Myo31DF antibody made in our laboratory previously (Fig. 14). In
these cells, I could distinguish *emc-*overexpressing cells from
*emc-*mutant cells by staining the Emc::GFP protein with an anti-GFP
antibody. I then compared the expression level and localization of
Myo31DF in these two types of cells. However, I did not find a marked
difference in the expression level and localization of Myo31DF between
*emc* mutant and *emc*-overexpressing cells (Fig. 14). These data
suggest that the expression level and localization of Myo31DF was not
controlled by *emc* or the downstream gene of *emc*. In addition, since
*DE-Cad* and *Myo61F* were also shown to be involved in the LR
asymmetric development of the embryonic hindgut (Hozumi et al., 2008;
Okumura et al., 2015; Taniguchi et al., 2011), I examined the expression
level of these genes in the *emc* mutant. However, again, I did not
observe statistically significant differences in the expression of these
genes between wild-type and *emc* mutant embryos (Fig. 15 and Fig. 16).
Thus, I have not yet identified the target(s) of Emc-Da that may
directly or indirectly interact with the Myo31DF protein in controlling
the LR asymmetry of the hindgut.

Formation of the dextral cell chirality depends on *da* suppression by *emc*
----------------------------------------------------------------------------

Our group previously demonstrated that chiral cell deformation drives
the anticlockwise rotation of the hindgut epithelial tube (Hatori et
al., 2014; Taniguchi et al., 2011). In the wild-type hindgut, before its
rotation (at stage 12), hindgut epithelial cells show cell chirality, in
which their apical cell boundaries tend to tilt leftwards (Fig. 17A)
(Hatori et al., 2014; Inaki et al., 2018a; Taniguchi et al., 2011).
Subsequent cancellation of this cell chirality leads to the hindgut
rotation (Fig. 17B) (Taniguchi et al., 2011). Our group also previously
showed that this cell chirality does not form properly in mutants that
show LR defects in their embryonic hindgut (Taniguchi et al., 2011).
Therefore, I suspected that the cell chirality does not form normally in
*emc* mutants, leading to the LR defects of this organ. To quantify the
cell chirality, I obtained confocal microscopic images that showed the
focal plane corresponding the apical region of the hindgut epithelial
cells, detected by anti-DE-Cad antibody staining, as described
previously (Fig. 18A) (Taniguchi et al., 2011). I then calculated the
angle *θ* between the apical cell boundaries and the anterior-posterior
axis of the hindgut, and computed the chirality index for each embryo
using the following formula: chirality index = (*N~R~* – *N~L~*)/(*N~R~*
+ *N~L~*) (Fig. 18B). In this formula, *N~R~* and *N~L~* represent the
numbers of boundaries with *θ* values from 0 to 90° and from -90 to 0°,
respectively (Fig. 18B). In this analysis, the chirality index is less
than 0 if the apical cell boundaries tend to tilt to the left side of
the hindgut AP axis (Fig. 18B). Note that the distribution of chirality
index was closely approximated by a normal distribution not by a bimodal
distribution even if the cell chirality is not normally distributed,
because of the central limit theorem (Fischer, 2011). The apical cell
boundaries tilted leftward in the wild-type hindgut, for which the mean
chirality index was -0.11 ± 0.03 (dextral chirality) (Fig. 19 and Fig.
20A). In contrast, in the hindgut of the *Myo31DF^K2^* homozygote, the
cell chirality was the mirror image (sinistral) of that in wild type,
with a chirality index [[]{#OLE_LINK2 .anchor}]{#OLE_LINK1 .anchor}of
0.13 ± 0.03 (Fig. 19 and Fig. 20B). These results recapitulated our
previous findings that wild-type and *Myo31DF* mutant cells show dextral
and sinistral states of cell chirality, respectively (Hatori et al.,
2014; Taniguchi et al., 2011).

Next, I measured the cell chirality of the hindgut cells in the
*emc^AP6^* mutant. In the *emc^AP6^* homozygote the mean chirality index
was nearly 0, indicating that the cell chirality disappeared, and that
the shape of the apical cell boundaries was LR symmetric in the hindgut
epithelium on average (Fig. 19 and Fig. 20C). To examine whether *emc*
was necessary to generate the dextral cell chirality in the hindgut
epithelium, I specifically overexpressed *emc::GFP*, driven by *NP2432*,
in this tissue in the *emc^AP6^* homozygote. The cell chirality was
significantly restored in these embryos (*p* &lt; 0.01;
*NP2432&gt;emc::GFP* in *emc^AP6^* in Fig. 19 and Fig. 20D). Thus, I
concluded that *emc* is required for the formation of dextral cell
chirality in this tissue, which is responsible for the LR-asymmetric
development of the hindgut. Consistent with my finding that *da*
overexpression resulted in LR embryonic-hindgut defects similar to those
of *emc* mutants (Fig. 8 and Fig. 9B), I also found that the cell
chirality was abolished by *NP2432-*driven *da* overexpression in the
hindgut epithelium, in which the chirality index was 0.04 ± 0.02 (*p*
&lt; 0.01; *NP2432&gt;da* in Fig. 19 and Fig. 20E).

Since the LR defects of the embryonic hindgut associated with *emc*
mutation were effectively restored by overexpressing *Myo31DF* in the
hindgut epithelium (Fig. 11 and Fig. 12B, D), I speculated that the loss
of cell chirality in the hindgut epithelium of *emc* mutants could be
also restored by *Myo31DF* overexpression in this tissue. I first
confirmed that the tissue-specific overexpression of *UAS-Myo31DF::RFP*
in the hindgut epithelium, driven by *NP2432*, changed the cell
chirality from sinistral to dextral (chirality index: -0.06 ± 0.02) in
the *Myo31DF^K2^* mutant (*NP2432&gt;Myo31DF::RFP* in *Myo31DF^K2^* in
Fig. 19 and Fig. 20F) (Hatori et al., 2014; Taniguchi et al., 2011). I
then similarly overexpressed *UAS-Myo31DF::RFP* in the *emc^AP6^*
homozygote, and found that the loss of cell chirality associated with
the *emc^AP6^* mutation was also significantly restored to the dextral
state in these embryos, in which the cell chirality index changed from
0.01 ± 0.02 to -0.14 ± 0.04 (compare *NP2342&gt;Myo31DF::RFP* in
*emc^AP6^* and *emc^AP6^* in Fig. 19 and Fig. 20C, G). In the *emc*
mutants, I ascertained that hyperactivated *da* causes LR asymmetric
defects of the embryonic hindgut (Fig. 7). I therefore examined whether
the cell chirality defects in the *emc* mutants could be rescued in the
absence of *da* function. *da^10^; emc^AP6^* double homozygous embryos
showed the wild-type (dextral) cell chirality in the hindgut epithelial
cells (chirality index: - 0.11 ± 0.02; Fig. 19). This data implicates
that the regulation of Da by Emc is required for the normal cell
chirality formation. If Da intermediates the cell chirality formation by
Emc, the cell chirality defects in the *da*-overexpressing embryos could
be rescued by co-overexpressing *Myo31DF* in the hindgut epithelium.
Therefore, I quantified the chirality index in the embryos
co-overexpressing *da* and *Myo31DF*. In the these embryos, the cell
chirality was restored from the achiral to the dextral state in this
tissue (chirality index: -0.12 ± 0.02; *NP2342&gt;da&gt; Myo31DF::RFP*
in Fig. 19 and Fig 20H). Taking these results together, I speculated
that Myo31DF functions downstream of or parallel to the regulatory
pathway of Emc and Da for the cell chirality formation and LR asymmetric
development of the hindgut (Fig 21).

Cell chirality sigmoidally correlates with the hindgut laterality
-----------------------------------------------------------------

Previously, our group suggested that dissolution of cell chirality
induces the rotation of the hindgut (Inaki, Hatori, et al., 2018;
Taniguchi et al., 2011). However, it was not studied well how much the
cell chirality contributes to the determination of the torsion direction
of the hindgut. To evaluate the potential contribution of the cell
chirality to the formation of the wild-type LR asymmetry, I analyzed the
correlation coefficient between these two properties. I analyzed the
relationships between the cell chirality and the probability of the
wild-type rotational direction in the hindgut (Fig. 22). The proportion
of hindguts demonstrating normal LR asymmetry that was calculated in
Fig. 2, Fig. 8, and Fig. 11 showed a strong negative correlation with
the chirality index that was calculated in Fig. 19 (R = - 0.926),
suggesting that dextral cell chirality tends to induce the normal LR
asymmetric development of the hindgut (Fig. 22). These observations
further suggested that the cell chirality controlled through Emc-Da and
Myo31DF is responsible for the LR asymmetric development of the
embryonic hindgut.

Although the cell chirality showed the strong correlation coefficient
with the frequency where the hindgut showed the wild-type LR asymmetry,
it was possible that the cell chirality non-linearly correlates with the
hindgut LR-asymmetry. To understand the relationship between these two
factors further, I elucidated another model that gives a better
coefficient of determination (R^2^). When I fitted the data by using a
linear function (![](media/image1.png){width="0.7180183727034121in"
height="0.17251093613298338in"}), the coefficient of determination was
0.858 (Fig. 22, and Table 3). On the other hand, the fitting by Hill
equation (![](media/image2.png){width="1.1046358267716536in"
height="0.29488188976377955in"}) gave the coefficient of determination
as 0.982 (Fig. 22, and Table 3) (Hill, 1910). This result means that the
Hill equation gives a better explanation for the relationships between
in the cell chirality and the LR laterality of the hindgut.

To further evaluate the contribution of the cell chirality to the LR
asymmetric rotation, I performed live imaging and elucidated whether
cell chirality in the living hindgut relates to the actual direction of
the rotation. I obtained the apical cell images of the hindgut before
rotation and continued culturing the embryo. After I cultured the embryo
for 3 hours, I evaluated the LR laterality of the hindgut. Because
wild-type embryos do not show the inverse or non-lateral phenotype of
the hindgut, I observed *emc^AP6^* embryos overexpressing
*UAS-myristoylated GFP* (*UAS-myrGFP*) driven by *byn-Gal4*, a hindgut
epithelium-specific driver. I calculated the chirality index of the
living embryos (live chirality index) as described above and analyzed
the relationship between the live chirality index and the LR laterality
of the hindgut in *emc^AP6^* mutant embryos. I evaluated the normal,
non-lateral, and inverse laterality as 1, 0.5, and 0, respectively (Fig.
23). Also in this analysis, the live chirality index showed a strong
negative correlation with the LR asymmetry of the hindgut (R = - 0.891)
(Fig. 23). I also fitted my data by using Hill equation and compared a
Hill equation model and a linear fitting model (Hill, 1910). The
coefficient of determination given by the fitted Hill equation were
0.872 and they were lower than those of the fitted linear model
(coefficient of determination of the linear model: 0.794) (Fig. 23, and
Table 3). This result indicated that the cell chirality in the living
cells also show sigmoidal correlation to the rotational direction of the
hindgut. These results suggested that the direction of the hindgut
rotation is determined stochastically following a distribution
approximated by Hill equation. Importantly, in the Hill equation, larger
*n* represents larger cooperativity of *x* to *y* (Hill, 1910). These
data implied that the cell chirality cooperatively determines the
rotational direction of the hindgut.

The apical cell surface of the hindgut epithelium expands in *emc* mutants, which is irrelevant to the cell chirality formation 
--------------------------------------------------------------------------------------------------------------------------------

My analysis of the apical cell boundaries in the hindgut epithelium also
revealed that the apical surface of these cells expanded in embryos
homozygous for *emc* mutations. Using the same images of the apical cell
boundaries in the hindgut epithelium before hindgut rotation (stage 12)
that were used for the cell-chirality analysis (as in Fig. 19A), I
measured the area of the apical surface of individual hindgut epithelial
cells. The area of the apical cell surface in the *emc^AP6^* mutant
(7.01±0.33 μm^2^) was about twice that of wild-type (2.85±0.29 μm^2^)
(Fig. 20A, C, and Fig. 24). This expansion of the apical surface in the
*emc^AP6^* mutant was fully suppressed by the tissue-specific
overexpression of *UAS-emc::GFP* in the hindgut epithelium driven by
*NP2432* (2.67 ± 0.23 μm^2^), suggesting that the lack of *emc*
functions in these cells was responsible for the apical expansion
(*NP2342&gt;emc::GFP* in *emc^AP6^* in Fig. 20D and Fig. 24). However,
in contrast to my observation that the cell chirality was abolished in
the *UAS-da*-overexpressing wild-type hindgut epithelium, the same
overexpression of *UAS-da* did not change the area of the apical surface
(*NP2342&gt;da* in Fig. 20E and Fig. 24). Therefore, I speculated that
*da* is not involved in the apical expansion, unlike the cell chirality
formation, of these cells in the *emc* mutant. To confirm this
hypothesis, I quantified the apical size of the hindgut cells in
*da^10^; emc^AP6^* double homozygous embryos. The apical area remained
larger in *da^10^; emc^AP6^* double mutants (2.67 ± 0.23 μm^2^; Fig.
24). These results collectively suggested that Emc has multiple actions
in the hindgut epithelium, and that while the cell chirality is probably
controlled through Da, the size of the apical surface is controlled
through a bHLH protein(s) other than Da (Fig. 21). This idea is
consistent with the ability of Id protein to form heterodimers with
various bHLH proteins (Benezra et al., 1990; Ling et al., 2014).

Considering that the overexpression of *da* abolished the cell chirality
but did not induce expansion of the apical surface in the hindgut
epithelial cells, I speculated that the apical expansion and the
disruption of cell chirality in these cells were independent events. To
test this possibility further, I examined whether the recovery of cell
chirality in the hindgut epithelial cells of *emc* mutants by the
overexpression of *Myo31DF* was coupled with a suppression of apical
expansion in these cells. I specifically overexpressed
*UAS-Myo31DF::RFP* in the hindgut epithelium of the *emc^AP6^*
homozygote, driven by *NP2432* and found that the size of the apical
surface was slightly rescued (from 7.01±0.33 μm^2^ in the *emc^AP6^*
homozygote to 5.48 ±0.30 μm^2^), but it was still significantly larger
than that of wild-type (*p* &lt; 0.001; 2.85±0.29 μm^2^)
(*NP2432&gt;Myo31DF::RFP* in *emc^AP6^* in Fig. 20G and Fig. 24). The
same overexpression of *UAS-Myo31DF::RFP* in the hindgut epithelium of
the *emc^AP6^* homozygote efficiently rescued the LR defects of the
embryonic hindgut associated with the *emc^AP6^* mutation, as mentioned
above (Fig. 11 and Fig 12E). Based on these analysis, I speculated that
the size of the apical surface does not affect the cell chirality
formation or the LR asymmetric rotation of the embryonic hindgut, which
is driven by the dissolution of the cell chirality (Taniguchi et al.,
2011). Furthermore, although the loss of cell chirality in the
*UAS-da*-overexpressing hindgut epithelium was thoroughly restored by
the co-overexpression of *UAS-Myo31DF::RFP* (Fig. 19), the size of the
apical surface was normal in the hindgut epithelium overexpressing
*UAS-da* alone (3.83 ± 0.35 μm^2^) or co-overexpressing *UAS-da* and
*UAS-Myo31DF::RFP* (3.98 ± 0.44 μm^2^) (*NP2432&gt;da* and
*NP2432&gt;da&gt;Myo31DF::RFP* in Fig. 24 and Fig. 20E, G). I also found
that the area of the apical surface was normal in the hindgut epithelium
of the *Myo31DF^K2^* homozygote (2.89 ± 0.16 μm^2^) (Fig. 20B and Fig.
24). Thus, I concluded that the expansion of the apical surface of the
hindgut epithelium in *emc* mutants was irrelevant to the disruption of
cell chirality formation and of the LR asymmetric rotation of the
embryonic hindgut (Fig 21).

It was also reported that cortical F-actin drives the apical
constriction of epithelial cells in various species (Heisenberg &
Bellaïche, 2013; Inoue et al., 2016; Lecuit & Lenne, 2007). Thus, I
speculated that the cortical localization of F-actin is disrupted in
*emc* mutants, which induces the apical expansion of the hindgut
epithelial cells. To examine this possibility, I generated
*UAS-emc::GFP*-overexpressing clonal cells on the *emc^AP6^* mutant
background, and stained the F-actin with a fluorescent phalloidin (Fig.
25). I compared the level of F-actin at the apical, lateral and basal
regions of the hindgut epithelial cells. However, I did not find a
marked difference in the level of phalloidin signal between these two
types of cells (Fig. 23), suggesting that F-actin was not involved in
the apical cell expansion caused by the lack of *emc* gene function.

The number of the hindgut epithelial cells is not changed in *emc* mutants
--------------------------------------------------------------------------

Our group previously reported that the reduction of cell number in the
hindgut epithelium disrupts the LR asymmetry of the hindgut (R. Maeda et
al., 2007; M. Nakamura et al., 2013). In *pebble* (*pbl*) and *string*
(*stg*) mutants, the proper cell proliferation was not underwent, which
inhibits the normal LR asymmetric development of the hindgut (M.
Nakamura et al., 2013). We also suggested that the properly regulated
cell number in the anal pad by *single-minded* (*sim*) might contribute
to the LR asymmetry of the embryonic hindgut (R. Maeda et al., 2007).
Because it was known that *emc* is involved in the cell proliferation
(Benezra et al., 1990; Lasorella et al., 2014; Wang & Baker, 2015a), I
suspected that the disruption of the cell number regulation induced the
disruption of the cell chirality and the LR asymmetry of the hindgut in
*emc* mutants. To confirm this hypothesis, I stained the nuclei with
anti-Lamin C antibody, and counted the number of Lamin C positive nuclei
in the hindgut. Previously, it was reported that Lamin C expression is
only detected in the hindgut during the *Drosophila* embryogenesis
(Riemer et al., 1995), thus the number of Lamin C positive nuclei
reflect the number of the hindgut cells (Fig. 26A, B). I compared the
number of the hindgut cells between wild-type and *emc* mutant and found
that the cell number was almost same (p = 0.717; wild-type: 687 ± 7,
*emc*: 694 ± 15; Fig. 26C). This data suggests that *emc* is not
required for the regulation of the cell number in the hindgut and that
the defects in the LR asymmetric development of the hindgut observed in
*emc* mutant is not due to the dysregulation of the cell proliferation.
Although I examined the cell properties such as the cell-type
specification, apical cell size, number, and cell chirality of hindgut
epithelial cells, only the phenotypes of the cell chirality correlate to
those the hindgut LR asymmetry. Based on these results, I concluded that
Emc-Da regulates cell chirality formation, in which Emc functions
upstream of or parallel to Myo31DF, and the proper chirality is involved
in the normal LR asymmetric rotation of the hindgut.

Chapter 6. Discussion\
======================

*da* needs to be suppressed by *emc* for the proper development of LR-asymmetry of the ***embryonic hindgut***
--------------------------------------------------------------------------------------------------------------

The regulatory network composed of E and Id proteins is known to control
a large number of cell-fate decisions, from *Drosophila* to humans (Ling
et al., 2014; Massari & Murre, 2000; Wang & Baker, 2015a). In this
study, I extended our understanding of this regulatory network by
revealing that the suppression of an E protein Da by an Id homolog Emc
is required for normal epithelial morphogenesis, in the LR asymmetric
rotation of the hindgut tube in
*Drosophila*.[]{#emc-and-da-modulation-in-lr-asymmetric-d .anchor} Emc
forms inactive heterodimers with various bHLH proteins, including Da, a
ubiquitous partner, and tissue-specific bHLH proteins, such as proneural
bHLH proteins (Ellis et al., 1990). I here identified *emc^tink^* as a
novel null allele of *emc* that disrupts the LR asymmetry of the
embryonic hindgut (Fig. 2 and Fig. 3). The deduced product of
*emc^tink^* lacks a Gln-rich region located at the C-terminus. This
Gln-rich region is conserved in various bHLH proteins, including Hairy,
in which it appears to be important for regulating the protein’s
degradation (Wainwright & Ish-Horowicz, 1992). My present results
suggest that the Gln-rich region in Id also has an essential role in its
suppression of Da (Fig. 4).

The LR defects of the hindgut associated with a null mutant of *emc*
were completely suppressed when combined with a *da* mutation (Fig. 8
and Fig. 9E), indicating that they could be completely attributed to the
hyperactivation of *da*. Consistent with this role of *emc* and *da* in
the embryonic hindgut, I also found that *emc* was expressed in the
embryonic hindgut before and during hindgut rotation (Fig. 5); the
ubiquitous expression of *da* during embryogenesis was previously
reported (Vaessin et al., 1994).

Although the hyperactivation of *da* was responsible for the LR defects
of the hindgut in the *emc* mutant (Fig 8 and Fig. 9B), I also found
that *da* was dispensable for the normal LR asymmetric development of
the hindgut (Fig. 8 and Fig. 9D). The hyperactivation of *da* was
harmful to this process, indicating that *da*’s function needs to be
suppressed by the wild-type *emc* to achieve the normal LR asymmetric
development. Da is known to regulate the transcription of many target
genes (Wang & Baker, 2015a). For example, the expression of *expanded*
(*ex*) is positively regulated through the binding of Da to the
regulatory region of the *ex* gene (Wang & Baker, 2015b). However, in
the present study I was unable to identify the target gene(s) of Da
responsible for the LR-asymmetric development of the embryonic gut.
Previously identified genes that are involved in the LR asymmetric
development of the embryonic hindgut include *DE-Cad* and *Myo31DF*
(Taniguchi et al., 2011). However, I was unable to detect alterations in
the expression of these genes in *emc* mutant embryos (Fig. 13, Fig. 14,
Fig. 15 and Fig. 16). To identify the genes responsible for inducing the
LR defects in *emc* mutants, I am currently analyzing alterations in the
expression level of genes in the *da*-overexpressing hindgut using an
RNA-seq approach.

The Emc-Da cascade is essential for the formation of dextral cell chirality
---------------------------------------------------------------------------

Our group previously demonstrated that the dissolution of cell chirality
in the hindgut epithelial cells is responsible for the LR-directional
rotation of this organ (Hatori et al., 2014; Inaki et al., 2016, 2018a;
Taniguchi et al., 2011). In agreement with this process, I here found
that cell chirality was not observed in the hindgut epithelium of the
*emc* mutant (Fig. 19 and Fig. 20C). I also found a lack of cell
chirality in *da*-overexpressing hindgut epithelial cells, consistent
with my idea that the hyperactivation of *da* in the hindgut epithelial
cells causes the LR defects in the *emc* mutant (Fig. 19 and Fig. 20E).
Therefore, I speculate that at least a downstream target gene of Da is
involved in the formation of cell chirality, although such target genes
currently remain unknown. However, importantly, I revealed that the
overexpression of *Myo31DF* in the hindgut epithelial cells of *emc*
mutant embryos restored both their cell chirality and the LR asymmetric
development of the hindgut (Fig. 11, Fig. 12E, Fig. 19, and Fig. 20G).
Thus, the crucial defects responsible for the failure in cell chirality
formation associated with the loss of *emc* function or the
hyperactivation of *da* in these epithelial cells could be compensated
for by an excess amount of Myo31DF. Similarly, the LR defects of the
hindgut and male genitalia in an *Abd-B* *Drosophila* mutant, in which
the default state of LR asymmetry is disrupted, are restored by
overexpressing *Myo31DF* (Coutelis et al., 2013). In addition, my
analysis of genetic epistasis between *emc* and *Myo31DF* in the cell
chirality formation and LR asymmetric development of the hindgut
revealed that Myo31DF functions downstream of or parallel to Emc. Thus,
the target gene(s) of Da encodes a protein that presumably has a close
relationship with Myo31DF biochemically, although this part of the
pathway remains unknown at this point.[[]{#cell-chirality-1
.anchor}]{#identification-of-the-novel-non-canonica .anchor}

Cell chirality may cooperatively determine the rotational direction of the hindgut
----------------------------------------------------------------------------------

Our group previously suggested that the dissolution of the cell
chirality generates mechanical force, which induced the rotation of the
hindgut (Hatori et al., 2014; Inaki, Hatori, et al., 2018; Taniguchi et
al., 2011). According to this hypothesis, an embryo showing the inverse
cell chirality to wild-type will show the inverse LR laterality of the
hindgut to the wild-type embryos' hindgut. In fact, in *Myo31DF*mutants,
the cell chirality is inverse to the wild-type, and the hindgut rotates
to the direction inverse to the wild-type (Fig. 22) (Hozumi et al.,
2006; Taniguchi et al., 2011). This hypothesis predicts that the hindgut
in which the cell chirality disappears cannot rotate to both directions
or weakly rotate to either side. However, despite the chirality index
was nearly 0, about 70% and 20% of the *emc* mutant embryos completely
rotated the hindgut to the wild-type direction and the inverse
direction, respectively (Fig. 2, 19A, and 20C). Based on these results,
I supposed that the rotational state of the hindgut is energetically
bistable where the unrotated hindgut is relatively unstable compared
with the rotated hindgut (Fig. 27). This hypothesis implies that the
hindgut where the cell chirality disappears is unstable and small
perturbations in individual embryos determine the rotational direction
in *emc* mutants (Fig. 27A, B’). On the other hand, because wild-type
cells have the enough cell chirality to determine the rotational
direction, major portion of the hindgut in wild-type embryos might
rotate to the wild-type direction (Fig. 27A, B).

This bi-state transition can be observed in a cooperative system such as
kinetics of allosteric enzymes (Monod et al., 1965). For example, it was
well known that hemoglobin-oxygen binding enhances the binding ability
of hemoglobin, which is required for the transition between high
affinity state and low affinity state of hemoglobin (Royer et al.,
2001). I supposed that such a cooperative translating system functions
in the hindgut to determine the LR asymmetry, because cooperativity was
observed in the relationship between the cell chirality and the
rotational direction of the hindgut (Fig. 22 and 23). In wild-type
or *Myo31DF*, major cells have cell chirality, which provides stable
determination of the rotational direction of the hindgut. On the other
hand, *emc* mutants lost the cell chirality of the hindgut cells on
average, which is in a boundary condition to determine the rotational
direction. In the boundary condition, small fluctuation of cell
chirality in individual embryos may stochastically determine the
laterality of the hindgut. At this point, I could not identify the
molecular entities or genes involved in this cooperative translating
system. To reveal the molecular basis of the translating system of cell
chirality must be important to understand how the LR asymmetric
laterality is determined in *Drosophila*.

*emc* controls the size of the apical cell surface independently of *da*
------------------------------------------------------------------------

I here found that wild-type *emc* reduced the area of the apical cell
surface of the hindgut epithelial cells at stage 12, given that this
area was significantly increased in *emc* mutant embryos, and this
increase was efficiently suppressed by the overexpression of *emc* in
the hindgut epithelium of these embryos (Fig. 20A, C, D, and Fig 24). In
contrast, the overexpression of *da* in the hindgut epithelium did not
affect the size of the apical cell surface (Fig. 20E, and Fig 24). This
result was unexpected, since both the *emc* mutant embryos and
*da*-overexpressing embryos showed similar defects in cell chirality
formation and LR asymmetric development of the hindgut. Therefore, the
expansion of the apical cell surface was not required for the LR defects
in the embryonic hindgut or the loss of cell chirality formation. In
addition, these results demonstrated that the wild-type *emc* gene
controlled the size of the apical surface independently of *da* gene
functions. This idea was consistent with the previous findings that Emc
forms heterodimers with various bHLH proteins to suppress their
transcription activities (Cabrera & Alonso, 1991; Garrell & Modolell,
1990; Murre et al., 1989). Various factors controlling the size of the
apical surface have been identified (Heisenberg & Bellaïche, 2013; Inoue
et al., 2016; Lecuit & Lenne, 2007). For example, the regulation of
cortical F-actin is involved in the apical constriction of epithelial
cells (Heisenberg & Bellaïche, 2013; Inoue et al., 2016; Lecuit & Lenne,
2007). However, in the hindgut epithelium of *emc* mutant cells, I did
not observe a marked alteration in the amount or distribution of the
cortical F-actin (Fig. 25). Therefore, the molecular mechanisms of the
apical expansion are unknown at this point.

As discussed above, the tissue-specific overexpression of *Myo31DF* in
the hindgut epithelium rescued the disappearance of cell chirality found
in the epithelial cells of the *emc* mutant hindgut. In contrast, the
same overexpression of *Myo31DF* did not completely rescue the increase
in the apical size of these cells, although *Myo31DF* overexpression
slightly reduced it (Fig. 20B, F, and Fig. 24). These results imply that
the restoration of dextral cell chirality by *Myo31DF* overexpression
was independent of the apical expansion under the *emc* mutant
condition. Although the expansion of the apical surface may influence
the mechanical conditions of the hindgut epithelium, my results showed
that the formation of cell chirality was not affected by it. In
addition, the dextral rotation of the hindgut was little affected by the
apical expansion. Therefore, the potential mechanical changes due to
apical expansion did not obstruct the LR directional rotation of the
hindgut.

Implication for the roles of cascades involving Id and E proteins in human diseases
-----------------------------------------------------------------------------------

*E* and *Id* genes are reported to be involved in development and
morphogenesis by controlling cell proliferation and cell differentiation
(Benezra et al., 1990; Ling et al., 2014; Massari & Murre, 2000; Wang &
Baker, 2015a). For example, mutant mice of *TCF4*, which encodes the
E2-2 protein, show defects in the hindbrain development, and *Id1* and
*Id3* mutant mice show cranial bone deformation and vascular
malformations in the brain(Flora et al., 2007; Lyden et al., 1999; Y.
Maeda et al., 2004). In *Drosophila*, a mosaic analysis in the imaginal
discs revealed that *emc* mutation leads to abnormal
tissue-specifications in the adult wings, legs, halteres, and notum
(Alonso & García -Bellido, 1988; de Celis et al., 1995). In addition,
dysregulations of E-Id cascades are responsible for various human
diseases (Benezra et al., 1990; Lasorella et al., 2014; Ling et al.,
2014; Wang & Baker, 2015a). For example, *TCF4* is a causal gene of
schizophrenia or Pitt-Hopkins syndrome in humans (Zweier et al., 2007).
An E-Id cascade is also disrupted in Diamond-Blackfan anemia patients,
implying that human bHLH is associated with bone marrow development
(Zhang et al., 1997). Thus, our current understanding of the pathology
of human diseases associated with aberrations of the E-Id cascade has
been mostly restricted to defects in cell-fate decisions including the
control of cell death and cell proliferation (Lasorella et al., 2014;
Massari & Murre, 2000; Wang & Baker, 2015a). In contrast, in this study,
I revealed that the E-Id cascade also regulates specific properties of
cell shape. The suppression of an E protein (Da) by Id (Emc) was
essential for the formation of dextral cell chirality, and the Id
protein was also required for apical constriction (Fig. 21). Therefore,
my results indicate that cell-shape change, which may drive tissue
deformation, should also be considered a possible outcome of the defects
in E-Id cascades (Fig. 28). Local cell shape plays an important role in
organ morphogenesis (Heisenberg & Bellaïche, 2013; Inoue et al., 2016;
Lecuit & Lenne, 2007; Leptin, 2005). Thus, my results suggest that
disorders in tissue and organ structure should be considered as
potential pathologies in diseases associated with defective E-Id
cascades.[]{#references .anchor}

Chapter 7. References\
======================

Adam, G., Perrimon, N., Noselli, S. (2003). The retinoic-like juvenile
hormone controls the looping of left-right asymmetric organs in
Drosophila. *Development*, 130:2397–2406.
http://doi.org/10.1242/dev.00460

Alonso, L. A. G., & García -Bellido, A. (1988). Extramacrochaetae, a
trans-acting gene of the achaete-scute complex of Drosophila involved in
cell communication. *Roux’s Archives of Developmental Biology*,
197:328–338. http://doi.org/10.1007/BF00375952

Arendt, D., & Nübler-Jung, K. (1997). Dorsal or ventral: Similarities in
fate maps and gastrulation patterns in annelids, arthropods and
chordates. *Mechanisms of Development*.
http://doi.org/10.1016/S0925-4773(96)00620-X

Asami, T., Gittenberger, E., & Falkner, G. (2008). Whole-body
enantiomorphy and maternal inheritance of chiral reversal in the pond
snail lymnaea stagnalis. *Journal of Heredity*.
http://doi.org/10.1093/jhered/esn032

Bai, J., Uehara, Y., & Montell, D. J. (2000). Regulation of invasive
cell behavior by taiman, a Drosophila protein related to AIB1, a steroid
receptor coactivator amplified in breast cancer. *Cell*, 103:1047–1058.
http://doi.org/10.1016/S0092-8674(00)00208-7

Baonza, A., de Celis, J. F., & García-Bellido, A. (2000). Relationships
between extramacrochaetae and Notch signalling in Drosophila wing
development. *Development (Cambridge, England)*, 127:2383–93. Retrieved
from http://www.ncbi.nlm.nih.gov/pubmed/10804180

Belle, I., & Zhuang, Y. (2014). E proteins in lymphocyte development and
lymphoid diseases. *Current Topics in Developmental Biology*,
110:153–187. http://doi.org/10.1016/B978-0-12-405943-6.00004-X

Benezra, R., Davis, R. L., Lockshon, D., Turner, D. L., & Weintraub, H.
(1990). The protein Id: A negative regulator of helix-loop-helix DNA
binding proteins. *Cell*, 61:49–59.
http://doi.org/10.1016/0092-8674(90)90214-Y

Bergmann, D. C. (2003). Embryonic handedness choice in C. elegans
involves the G  protein GPA-16. *Development*.
http://doi.org/10.1242/dev.00839

Berleth, T., Burri, M., Thoma, G., Bopp, D., Richstein, S., Frigerio,
G., … Nüsslein-Volhard, C. (1988). The role of localization of bicoid
RNA in organizing the anterior pattern of the Drosophila embryo. *The
EMBO Journal*, 7:1749–1756.
http://doi.org/10.1002/j.1460-2075.1988.tb03004.x

Bhattacharya, A., & Baker, N. E. (2011). A network of broadly expressed
hlh genes regulates tissue-specific cell fates. *Cell*, 147:881–892.
http://doi.org/10.1016/j.cell.2011.08.055

Blochlinger, K., Bodmer, R., Jack, J., Jan, L. Y., & Jan, Y. N. (1988).
Primary structure and expression of a product from cut, a locus involved
in specifying sensory organ identity in Drosophila. *Nature*,
333:629–635. http://doi.org/10.1038/333629a0

Blum, M., & Ott, T. (2018). Animal left–right asymmetry. *Current
Biology*. http://doi.org/10.1016/j.cub.2018.02.073

Burdine, R. D., & Schier, A. F. (2000). Conserved and divergent
mechanisms in left-right axis formation. *Genes and Development*.
http://doi.org/10.1101/gad.14.7.763

Cabrera, C. V, & Alonso, M. C. (1991). Transcriptional activation by
heterodimers of the achaete-scute and daughterless gene products of
*Drosophila*. *The EMBO Journal*, 10:2965–73. Retrieved from
http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=453011&tool=pmcentrez&rendertype=abstract

Campos-Ortega, J. A. (2015). *\[Book\] The Embryonic Development of
Drosophila Melanogaster*. *Statewide Agricultural Land Use Baseline
2015* (Vol. 1). http://doi.org/10.1007/978-3-662-02454-6

Caudy, M., Grell, E. H., Dambly-Chaudière, C., Ghysen, A., Jan, L. Y., &
Jan, Y. N. (1988). The maternal sex determination gene daughterless has
zygotic activity necessary for the formation of peripheral neurons in
Drosophila. *Genes & Development*, 2:843–852.
http://doi.org/10.1101/gad.2.7.843

Caudy, M., Vässin, H., Brand, M., Tuma, R., Jah, L. Y., & Jan, Y. N.
(1988). daughterless, a Drosophila gene essential for both neurogenesis
and sex determination, has sequence similarities to myc and the
achaete-scute complex. *Cell*, 55:1061–1067.
http://doi.org/10.1016/0092-8674(88)90250-4

Chen, T. H., Hsu, J. J., Zhao, X., Guo, C., Wong, M. N., Huang, Y., …
Demer, L. L. (2012). Left-right symmetry breaking in tissue
morphogenesis via cytoskeletal mechanics. *Circulation Research*,
110:551–559. http://doi.org/10.1161/CIRCRESAHA.111.255927

Coutelis, J. B., Géminard, C., Spéder, P., Suzanne, M., Petzoldt, A. G.,
& Noselli, S. (2013). Drosophila Left/Right Asymmetry Establishment Is
Controlled by the Hox Gene Abdominal-B. *Developmental Cell*, 24:89–97.
http://doi.org/10.1016/j.devcel.2012.11.013

Coutelis, J. B., González-Morales, N., Géminard, C., & Noselli, S.
(2014). Diversity and convergence in the mechanisms establishing L/R
asymmetry in metazoa. *EMBO Reports*, 15:926–37.
http://doi.org/10.15252/embr.201438972

Crews, S. T., Thomas, J. B., & Goodman, C. S. (1988). The Drosophila
single-minded gene encodes a nuclear protein with sequence similarity to
the per gene product. *Cell*, 52:143–151.
http://doi.org/10.1016/0092-8674(88)90538-7

Cronmiller, C., Schedl, P., & Cline, T. W. (1988). Molecular
characterization of daughterless, a Drosophila sex determination gene
with multiple roles in development. *Genes & Development*, 2:1666–1676.
http://doi.org/10.1101/gad.2.12a.1666

Cummings, C. a, & Cronmiller, C. (1994). The daughterless gene functions
together with Notch and Delta in the control of ovarian follicle
development in Drosophila. *Development (Cambridge, England)*,
120:381–94. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8149916

Davison, A., McDowell, G. S., Holden, J. M., Johnson, H. F.,
Koutsovoulos, G. D., Liu, M. M., … Blaxter, M. L. (2016). Formin is
Associated with Left-Right Asymmetry in the Pond Snail and the Frog.
*Current Biology*, 26:654–660. http://doi.org/10.1016/j.cub.2015.12.071

de Celis, J. F., Baonza, A., & García-Bellido, A. (1995). Behavior of
extramacrochaetae mutant cells in the morphogenesis of the Drosophila
wing. *Mechanisms of Development*, 53:209–221.
http://doi.org/10.1016/0925-4773(95)00436-5

Ellis, H. M. (1994). Embryonic expression and function of the Drosophila
helix-loop-helix gene, extramacrochaetae. *Mechanisms of Development*,
47:65–72. http://doi.org/10.1016/0925-4773(94)90096-5

Ellis, H. M., Spann, D. R., & Posakony, J. W. (1990). extramacrochaetae,
a negative regulator of sensory organ development in Drosophila, defines
a new class of helix-loop-helix proteins. *Cell*, 61:27–38.
http://doi.org/10.1016/0092-8674(90)90212-W

Fischer, H. (2011). *A History of the Central Limit Theorem*. *Media*.
http://doi.org/10.1007/978-0-387-87857-7

Flora, A., Garcia, J. J., Thaller, C., & Zoghbi, H. Y. (2007). The
E-protein Tcf4 interacts with Math1 to regulate differentiation of a
specific subset of neuronal progenitors. *Proceedings of the National
Academy of Sciences*, 104:15382–15387.
http://doi.org/10.1073/pnas.0707456104

Fuß, B., & Hoch, M. (2002). Notch signaling controls cell fate
specification along the dorsoventral axis of the Drosophila gut.
*Current Biology*, 12:171–179.
http://doi.org/10.1016/S0960-9822(02)00653-X

Garrell, J., & Modolell, J. (1990). The Drosophila extramacrochaetae
locus, an antagonist of proneural genes that, like these genes, encodes
a helix-loop-helix protein. *Cell*, 61:39–48.
http://doi.org/10.1016/0092-8674(90)90213-X

Giebel, B., Stüttem, I., Hinz, U., & Campos-Ortega, J. A. (1997). Lethal
of Scute requires overexpression of Daughterless to elicit ectopic
neuronal development during embryogenesis in Drosophila. *Mechanisms of
Development*, 63:75–87. http://doi.org/10.1016/S0925-4773(97)00029-4

González-Morales, N., Géminard, C., Lebreton, G., Cerezo, D., Coutelis,
J. B., & Noselli, S. (2015). The Atypical Cadherin Dachsous Controls
Left-Right Asymmetry in Drosophila. *Developmental Cell*, 33:675–689.
http://doi.org/10.1016/j.devcel.2015.04.026

Grabowski, D. T., Carney, J. P., & Kelley, M. R. (1991). A drosophila
gene containing the opa repetitive element is exclusively expressed in
adult male abdomens. *Nucleic Acids Research*.
http://doi.org/10.1093/nar/19.7.1709

Gyoja, F. (2017). Basic helix-loop-helix transcription factors in
evolution: Roles in development of mesoderm and neural tissues.
*Genesis*. http://doi.org/10.1002/dvg.23051

Hamada, H., Meno, C., Watanabe, D., & Saijoh, Y. (2002). Establishment
of vertebrate left-right asymmetry. *Nature Reviews Genetics*.
http://doi.org/10.1038/nrg732

Hatori, R., Ando, T., Sasamura, T., Nakazawa, N., Nakamura, M.,
Taniguchi, K., … Matsuno, K. (2014). Left-right asymmetry is formed in
individual cells by intrinsic cell chirality. *Mechanisms of
Development*, 133:146–162. http://doi.org/10.1016/j.mod.2014.04.002

Hayashi, S., Ito, K., Sado, Y., Taniguchi, M., Akimoto, A., Takeuchi,
H., … Goto, S. (2002). GETDB, a database compiling expression patterns
and molecular locations of a collection of gal4 enhancer traps.
*Genesis*, 34:58–61. http://doi.org/10.1002/gene.10137

Hayashi, T., & Murakami, R. (2001). Left-right asymmetry in Drosophila
melanogaster gut development. *Development Growth and Differentiation*,
43:239–246. http://doi.org/10.1046/j.1440-169X.2001.00574.x

Heisenberg, C. P., & Bellaïche, Y. (2013). XForces in tissue
morphogenesis and patterning. *Cell*, 153.
http://doi.org/10.1016/j.cell.2013.05.008

Hill, A. V. (1910). The possible effects of the aggregation of the
molecules of haemoglobin on its dissociation curves. *Proceedings of the
Physiological Society*, 15. http://doi.org/10.1017/CBO9781107415324.004

Hirokawa, N., Tanaka, Y., Okada, Y., & Takeda, S. (2006). Nodal Flow and
the Generation of Left-Right Asymmetry. *Cell*.
http://doi.org/10.1016/j.cell.2006.03.002

Hoch, M., & Pankratz, M. J. (1996). Control of gut development by fork
head and cell signaling molecules in Drosophila. *Mechanisms of
Development*, 58:3–14. http://doi.org/10.1016/S0925-4773(96)00541-2

Hozumi, S., Maeda, R., Taniguchi-Kanai, M., Okumura, T., Taniguchi, K.,
Kawakatsu, Y., … Matsuno, K. (2008). Head region of unconventional
myosin I family members is responsible for the organ-specificity of
their roles in left-right polarity in Drosophila. *Developmental
Dynamics*, 237:3528–3537. http://doi.org/10.1002/dvdy.21583

Hozumi, S., Maeda, R., Taniguchi, K., Kanai, M., Shirakabe, S.,
Sasamura, T., … Matsuno, K. (2006). An unconventional myosin in
Drosophila reverses the default handedness in visceral organs. *Nature*,
440:798–802. http://doi.org/10.1038/nature04625

Inaki, M., Hatori, R., Nakazawa, N., Okumura, T., Ishibashi, T., Kikuta,
J., … Honda, H. (2018). Chiral cell sliding drives left-right asymmetric
organ twisting. *ELife*, 7. http://doi.org/10.7554/eLife.32506

Inaki, M., Liu, J., & Matsuno, K. (2016). Cell chirality: Its origin and
roles in left-right asymmetric development. *Philosophical Transactions
of the Royal Society B: Biological Sciences*.
http://doi.org/10.1098/rstb.2015.0403

Inaki, M., Sasamura, T., & Matsuno, K. (2018). Cell Chirality Drives
Left-Right Asymmetric Morphogenesis. *Frontiers in Cell and
Developmental Biology*, 6. http://doi.org/10.3389/fcell.2018.00034

Inoue, Y., Suzuki, M., Watanabe, T., Yasue, N., Tateo, I., Adachi, T., &
Ueno, N. (2016). Mechanical roles of apical constriction, cell
elongation, and cell migration during neural tube formation in Xenopus.
*Biomechanics and Modeling in Mechanobiology*, 15:1733–1746.
http://doi.org/10.1007/s10237-016-0794-1

Iwaki, D. D., Johansen, K. A., Singer, J. B., & Lengyel, J. A. (2001).
Drumstick, bowl, and lines are required for patterning and cell
rearrangement in the Drosophila embryonic hindgut. *Developmental
Biology*, 240:611–626. http://doi.org/10.1006/dbio.2001.0483

Juan, T., Géminard, C., Coutelis, J. B., Cerezo, D., Polès, S., Noselli,
S., & Fürthauer, M. (2018). Myosin1D is an evolutionarily conserved
regulator of animal left-right asymmetry. *Nature Communications*, 9.
http://doi.org/10.1038/s41467-018-04284-8

Kassis, J. A., Poole, S. J., Wright, D. K., & O ’farrell, P. H. (1986).
Sequence conservation in the protein coding and intron regions of the
engrailed transcription unit. *The EMBO Journal*, 20:3583–3589.
http://doi.org/10.1007/s00421-011-1977-1

Kidd, S., Kelley, M. R., & Young, M. W. (1986). Sequence of the Notch
Locus of Drosophila melanogaster: Relationship of the Encoded Protein to
Mammalian Clotting and Growth Factors. *Molecular and Cellular Biology*,
6:3094–3108. http://doi.org/10.1128/MCB.6.9.3094.Updated

Kimelman, D., & Martin, B. L. (2012). Anterior-posterior patterning in
early development: Three strategies. *Wiley Interdisciplinary Reviews:
Developmental Biology*, 1:253–266. http://doi.org/10.1002/wdev.25

Kosman, D., Ip, Y. T., Levine, M., & Arora, K. (1991). Establishment of
the mesoderm-neuroectoderm boundary in the Drosophila embryo. *Science*,
254:118–122. http://doi.org/10.1126/science.1925551

Kuroda, J., Nakamura, M., Yoshida, M., Yamamoto, H., Maeda, T.,
Taniguchi, K., … Matsuno, K. (2012). Canonical Wnt signaling in the
visceral muscle is required for left-right asymmetric development of the
Drosophila midgut. *Mechanisms of Development*, 128:625–639.
http://doi.org/10.1016/j.mod.2011.12.002

Kuroda, R., Endo, B., Abe, M., & Shimizu, M. (2009). Chiral blastomere
arrangement dictates zygotic left-right asymmetry pathway in snails.
*Nature*, 462:790–794. http://doi.org/10.1038/nature08597

Lasorella, A., Benezra, R., & Iavarone, A. (2014). The ID proteins:
Master regulators of cancer stem cells and tumour aggressiveness.
*Nature Reviews Cancer*. http://doi.org/10.1038/nrc3638

Laughon, A., Boulet, A. M., Bermingham Jr., J. R., Laymon, R. A., &
Scott, M. P. (1986). Structure of transcripts from the homeotic
Antennapedia gene of Drosophila melanogaster: two promoters control the
major protein-coding region. *Mol Cell Biol*, 6:4676–4689.
http://doi.org/10.1128/MCB.6.12.4676

Lecuit, T., & Lenne, P.-F. (2007). Cell surface mechanics and the
control of cell shape, tissue patterns and morphogenesis. *Nature
Reviews. Molecular Cell Biology*, 8:633–644.
http://doi.org/10.1038/nrm2222

Lengyel, J. A., & Iwaki, D. D. (2002). It takes guts: The Drosophila
hindgut as a model system for organogenesis. *Developmental Biology*.
http://doi.org/10.1006/dbio.2002.0577

Leptin, M. (2005). Gastrulation movements: The logic and the nuts and
bolts. *Developmental Cell*, 8:305–320.
http://doi.org/10.1016/j.devcel.2005.02.007

Levin, M. (2005). Left-right asymmetry in embryonic development: A
comprehensive review. *Mechanisms of Development*.
http://doi.org/10.1016/j.mod.2004.08.006

Ligoxygakis, P., Strigini, M., & Averof, M. (2001). Specification of
left-right asymmetry in the embryonic gut of Drosophila. *Development*,
128:1171–4. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/11245582

Ling, F., Kang, B., & Sun, X. H. (2014). Id proteins: Small molecules,
mighty regulators. *Current Topics in Developmental Biology*,
110:189–216. http://doi.org/10.1016/B978-0-12-405943-6.00005-1

Lyden, D., Young, A. Z., Zagzag, D., Yan, W., Gerald, W., O’Reilly, R.,
… Benezra, R. (1999). Id1 and Id3 are required for neurogenesis,
angiogenesis and vascularization of tumour xenografts. *Nature*,
401:670–677. http://doi.org/10.1038/44334

Maeda, R., Hozumi, S., Taniguchi, K., Sasamura, T., Murakami, R., &
Matsuno, K. (2007). Roles of single-minded in the left-right asymmetric
development of the Drosophila embryonic gut. *Mechanisms of
Development*, 124:204–217. http://doi.org/10.1016/j.mod.2006.12.001

Maeda, Y., Tsuji, K., Nifuji, A., & Noda, M. (2004). Inhibitory
helix-loop-helix transcription factors Id1/Id3 promote bone formation in
vivo. *Journal of Cellular Biochemistry*, 93:337–344.
http://doi.org/10.1002/jcb.20154

Massari, M. E., Grant, P. A., Pray-Grant, M. G., Berger, S. L., Workman,
J. L., & Murre, C. (1999). A conserved motif present in a class of
helix-loop-helix proteins activates transcription by direct recruitment
of the SAGA complex. *Molecular Cell*, 4:63–73.
http://doi.org/10.1016/S1097-2765(00)80188-4

Massari, M. E., & Murre, C. (2000). Helix-Loop-Helix Proteins:
Regulators of Transcription in Eucaryotic Organisms. *Molecular and
Cellular Biology*, 20:429–440.
http://doi.org/10.1128/MCB.20.2.429-440.2000

McGinnis, W., Levine, M. S., Hafen, E., Kuroiwa, A., & Gehring, W. J.
(1984). A conserved DNA sequence in homoeotic genes of the Drosophila
Antennapedia and bithorax complexes. *Nature*, 308:428–433.
http://doi.org/10.1038/308428a0

McGrath, J., Somlo, S., Makova, S., Tian, X., & Brueckner, M. (2003).
Two populations of node monocilia initiate left-right asymmetry in the
mouse. *Cell*, 114:61–73. http://doi.org/10.1016/S0092-8674(03)00511-7

Monod, J., Wyman, J., & Changeux, J. P. (1965). On the nature of
allosteric transitions: A plausible model. *Journal of Molecular
Biology*, 12:88–118. http://doi.org/10.1016/S0022-2836(65)80285-6

Morgenstern, B., & Atchley, W. R. (1999). Evolution of bHLH
transcription factors: Modular evolution by domain shuffling? *Molecular
Biology and Evolution*, 16:1654–1663.
http://doi.org/10.1093/oxfordjournals.molbev.a026079

Murre, C. (2005). Helix-loop-helix proteins and lymphocyte development.
*Nature Immunology*. http://doi.org/10.1038/ni1260

Murre, C., McCaw, P. S., Vaessin, H., Caudy, M., Jan, L. Y., Jan, Y. N.,
… Baltimore, D. (1989). Interactions between heterologous
helix-loop-helix proteins generate complexes that bind specifically to a
common DNA sequence. *Cell*, 58:537–544.
http://doi.org/10.1016/0092-8674(89)90434-0

Naganathan, S. R. am, Fürthauer, S., Nishikawa, M., Jülicher, F., &
Grill, S. W. (2014). Active torque generation by the actomyosin cell
cortex drives left-right symmetry breaking. *ELife*, 3:e04165.
http://doi.org/10.7554/eLife.04165

Nakamura, M., Matsumoto, K., Iwamoto, Y., Muguruma, T., Nakazawa, N.,
Hatori, R., … Matsuno, K. (2013). Reduced cell number in the hindgut
epithelium disrupts hindgut left-right asymmetry in a mutant of pebble,
encoding a RhoGEF, in Drosophila embryos. *Mechanisms of Development*,
130:169–180. http://doi.org/10.1016/j.mod.2012.09.007

Nakamura, T., & Hamada, H. (2012). Left-right patterning: conserved and
divergent mechanisms. *Development*, 139:3257–3262.
http://doi.org/10.1242/dev.061606

Nakazawa, N., Taniguchi, K., Okumura, T., Maeda, R., & Matsuno, K.
(2012). A novel Cre/loxP system for mosaic gene expression in the
Drosophila embryo. *Developmental Dynamics*, 241:965–974.
http://doi.org/10.1002/dvdy.23784

Nam, H. song, & Benezra, R. (2009). High Levels of Id1 Expression Define
B1 Type Adult Neural Stem Cells. *Cell Stem Cell*, 5:515–526.
http://doi.org/10.1016/j.stem.2009.08.017

Niola, F., Zhao, X., Singh, D., Castano, A., Sullivan, R., Lauria, M., …
Lasorella, A. (2012). Id proteins synchronize stemness and anchorage to
the niche of neural stem cells. *Nature Cell Biology*, 14:477–487.
http://doi.org/10.1038/ncb2490

Okumura, T., Sasamura, T., Inatomi, M., Hozumi, S., Nakamura, M.,
Hatori, R., … Matsuno, K. (2015). Class I myosins have overlapping and
specialized functions in left-right asymmetric development in
Drosophila. *Genetics*, 199:1183–1199.
http://doi.org/10.1534/genetics.115.174698

Okumura, T., Utsuno, H., Kuroda, J., Gittenberger, E., Asami, T., &
Matsuno, K. (2008). The development and evolution of left-right
asymmetry in invertebrates: Lessons from Drosophila and snails.
*Developmental Dynamics*. http://doi.org/10.1002/dvdy.21788

Pagliuca, A., Gallo, P., De Luca, P., & Lania, L. (2000). Class A
helix-loop-helix proteins are positive regulators of several
cyclin-dependent kinase inhibitors’ promoter activity and negatively
affect cell growth. *Cancer Research*, 60:1376–1382.

Pascual, A., Huang, K. L., Neveu, J., & Préat, T. (2004). Brain
asymmetry and long-term memory: Fruitflies that have structurally
similar brain hemispheres forget within a matter of hours. *Nature*,
427:605–606. http://doi.org/10.1038/427605a

Petzoldt, A. G., Coutelis, J.-B., Géminard, C., Spéder, P., Suzanne, M.,
Cerezo, D., & Noselli, S. (2012). DE-Cadherin regulates unconventional
Myosin ID and Myosin IC in Drosophila left-right asymmetry
establishment. *Development*, 139:1874–1884.

Pohl, C., & Bao, Z. (2010). Chiral forces organize left-right patterning
in C. elegans by uncoupling midline and anteroposterior axis.
*Developmental Cell*, 19:402–412.
http://doi.org/10.1016/j.devcel.2010.08.014

Popova, M. K., He, W., Korenjak, M., Dyson, N. J., & Moon, N.-S. (2011).
Rb deficiency during *Drosophila* eye development deregulates EMC,
causing defects in the development of photoreceptors and cone cells.
*Journal of Cell Science*, 124:4203–4212.
http://doi.org/10.1242/jcs.088773

Prabhu, S., Ignatova, A., Park, S. T., & Sun, X. H. (1997). Regulation
of the expression of cyclin-dependent kinase inhibitor p21 by E2A and Id
proteins. *Molecular and Cellular Biology*, 17:5888–5896.
http://doi.org/10.1128/MCB.17.10.5888

Regulski, M., Harding, K., Kostriken, R., Karch, F., Levine, M., &
McGinnis, W. (1985). Homeo box genes of the Antennapedia and Bithorax
Complexes of Drosophila. *Cell*, 43:71–80.
http://doi.org/10.1016/0092-8674(85)90013-3

Rice, B., & Liu, E. (2015). The Nature, Extent, and Consequences of
Genetic Variation in the opa Repeats of Notch in Drosophila. *G3
(Bethesda)*, 5:2405–2419. http://doi.org/10.1534/g3.115.021659

Riechmann, V., & Ephrussi, A. (2001). Axis formation during Drosophila
oogenesis. *Current Opinion in Genetics and Development*, 11:374–383.
http://doi.org/10.1016/S0959-437X(00)00207-0

Riemer, D., Stuurman, N., Berrios, M., Hunter, C., Fisher, P. A., &
Weber, K. (1995). Expression of Drosophila lamin C is developmentally
regulated: analogies with vertebrate A-type lamins. *J Cell Sci*,
108:3189–3198.

Royer, W. E., Knapp, J. E., Strand, K., & Heaslet, H. A. (2001).
Cooperative hemoglobins: Conserved fold, diverse quaternary assemblies
and allosteric mechanisms. *Trends in Biochemical Sciences*.
http://doi.org/10.1016/S0968-0004(01)01811-4

Sato, K., Hiraiwa, T., Maekawa, E., Isomura, A., Shibata, T., &
Kuranaga, E. (2015). Left-right asymmetric cell intercalation drives
directional collective cell movement in epithelial morphogenesis.
*Nature Communications*, 6. http://doi.org/10.1038/ncomms10074

Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair,
M., Pietzsch, T., … Cardona, A. (2012). Fiji: An open-source platform
for biological-image analysis. *Nature Methods*.
http://doi.org/10.1038/nmeth.2019

Shibazaki, Y., Shimizu, M., & Kuroda, R. (2004). Body handedness is
directed by genetically determined cytoskeletal dynamics in the early
embryo. *Current Biology*. http://doi.org/10.1016/j.cub.2004.08.018

Sloan, S. R., Shen, C.-P., Mccarrick-Walmsley, R., & Kadesch, T. (1996).
Phosphorylation of E47 as a Potential Determinant of B-Cell-Specific
Activity. *Molecular and Cellular Biology*, 16:6900–6908.
http://doi.org/10.1128/MCB.16.12.6900

Smith, J., Cummings, C., & Cronmiller, C. (2002). Daughterless
coordinates somatic cell proliferation, differentiation and germline
cyst survival during follicle formation in Drosophila. *Development*,
3267:3255–3267. Retrieved from
http://dev.biologists.org/content/129/13/3255.short

Spéder, P., Ádám, G., & Noselli, S. (2006). Type ID unconventional
myosin controls left-right asymmetry in Drosophila. *Nature*,
440:803–807. http://doi.org/10.1038/nature04623

Steward, R. (1987). Dorsal, an embryonic polarity gene in drosophila, is
homologous to the vertebrate proto-oncogene, c-rel. *Science*,
238:692–694. http://doi.org/10.1126/science.3118464

Sullivan, W. A., Ashburner, M., & Hawley, R. S. (2000). *Drosophila
protocols.* *Cold Spring Harbor Laboratory*.
http://doi.org/10.1007/978-1-61779-034-8\_15

Tabin, C. J. (2006). The Key to Left-Right Asymmetry. *Cell*.
http://doi.org/10.1016/j.cell.2006.09.018

Tabin, C. J., & Vogan, K. J. (2003). A two-cilia model for vertebrate
left-right axis specification. *Genes and Development*.
http://doi.org/10.1101/gad.1053803

Takashima, S., & Murakami, R. (2001). Regulation of pattern formation in
the Drosophila hindgut by wg, hh, dpp, and en. *Mechanisms of
Development*, 101:79–90. http://doi.org/10.1016/S0925-4773(00)00555-4

Tamada, A., & Igarashi, M. (2017). Revealing chiral cell motility by 3D
Riesz transform-differential interference contrast microscopy and
computational kinematic analysis. *Nature Communications*, 8.
http://doi.org/10.1038/s41467-017-02193-w

Tamada, A., Kawase, S., Murakami, F., & Kamiguchi, H. (2010). Autonomous
right-screw rotation of growth cone filopodia drives neurite turning.
*Journal of Cell Biology*, 188:429–441.
http://doi.org/10.1083/jcb.200906043

Taniguchi, K., Hozumi, S., Maeda, R., Ooike, M., Sasamura, T., Aigaki,
T., & Matsuno, K. (2007). D-JNK signaling in visceral muscle cells
controls the laterality of the Drosophila gut. *Developmental Biology*,
311:251–263. http://doi.org/10.1016/j.ydbio.2007.08.048

Taniguchi, K., Maeda, R., Ando, T., Okumura, T., Nakazawa, N., Hatori,
R., … Matsuno, K. (2011). Chirality in planar cell shape contributes to
left-right asymmetric epithelial morphogenesis. *Science*, 333:339–341.
http://doi.org/10.1126/science.1200940

Tautz, D., Lehmann, R., Schnürch, H., Schuh, R., Seifert, E., Kienlin,
A., … Jäckle, H. (1987). Finger protein of novel structure encoded by
hunchback, a second member of the gap class of Drosophila segmentation
genes. *Nature*, 327:383–389. http://doi.org/10.1038/327383a0

Tautz, D., & Pfeifle, C. (1989). A non-radioactive in situ hybridization
method for the localization of specific RNAs in Drosophila embryos
reveals translational control of the segmentation gene hunchback.
*Chromosoma*, 98:81–85. http://doi.org/10.1007/BF00291041

Tee, Y. H., Shemesh, T., Thiagarajan, V., Hariadi, R. F., Anderson, K.
L., Page, C., … Bershadsky, A. D. (2015). Cellular chirality arising
from the self-organization of the actin cytoskeleton. *Nature Cell
Biology*, 17:445–457. http://doi.org/10.1038/ncb3137

Thisse, B., Stoetzel, C., Gorostiza-Thisse, C., & Perrin-Schmitt, F.
(1988). Sequence of the twist gene and nuclear localization of its
protein in endomesodermal cells of early Drosophila embryos. *The EMBO
Journal*, 7:2175–83. http://doi.org/10.1002/J.1460-2075.1988.TB03056.X

Tingler, M., Kurz, S., Maerker, M., Ott, T., Fuhl, F., Schweickert, A.,
… Blum, M. (2018). A Conserved Role of the Unconventional Myosin 1d in
Laterality Determination. *Current Biology*, 28:810–816.e3.
http://doi.org/10.1016/j.cub.2018.01.075

Vaessin, H., Brand, M., Jan, L. Y., & Jan, Y. N. (1994). daughterless is
essential for neuronal precursor differentiation but not for initiation
of neuronal precursor formation in Drosophila embryo. *Development*,
120:935–45. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/7600969

VanGuilder, H. D., Vrana, K. E., & Freeman, W. M. (2008). Twenty-five
years of quantitative PCR for gene expression analysis. *BioTechniques*.
http://doi.org/10.2144/000112776

Wainwright, S. M., & Ish-Horowicz, D. (1992). Point mutations in the
Drosophila hairy gene demonstrate in vivo requirements for basic,
helix-loop-helix, and WRPW domains. *Molecular and Cellular Biology*,
12:2475–83. http://doi.org/10.1128/MCB.12.6.2475.Updated

Wan, L. Q., Ronaldson, K., Park, M., Taylor, G., Zhang, Y., Gimble, J.
M., & Vunjak-Novakovic, G. (2011). Micropatterned mammalian cells
exhibit phenotype-specific left-right asymmetry. *Proceedings of the
National Academy of Sciences*, 108:12295–12300.
http://doi.org/10.1073/pnas.1103834108

Wang, L. H., & Baker, N. E. (2015a). E Proteins and ID Proteins:
Helix-Loop-Helix Partners in Development and Disease. *Developmental
Cell*. http://doi.org/10.1016/j.devcel.2015.10.019

Wang, L. H., & Baker, N. E. (2015b). Salvador-Warts-Hippo Pathway in a
Developmental Checkpoint Monitoring Helix-Loop-Helix Proteins.
*Developmental Cell*, 32:191–202.
http://doi.org/10.1016/j.devcel.2014.12.002

Wells, R. E., Barry, J. D., Warrington, S. J., Cuhlmann, S., Evans, P.,
Huber, W., … Zeidler, M. P. (2013). Control of tissue morphology by
Fasciclin III-mediated intercellular adhesion. *Development*,
140:3858–3868. http://doi.org/10.1242/dev.096214

Wharton, K. A., Yedvobnick, B., Finnerty, V. G., & Artavanis-Tsakonas,
S. (1985). opa: A novel family of transcribed repeats shared by the
Notch locus and other developmentally regulated loci in D. melanogaster.
*Cell*, 40:55–62. http://doi.org/10.1016/0092-8674(85)90308-3

Wülbeck, C., Fromental-Ramain, C., & Campos-Ortega, J. A. (1994). The
HLH domain of a zebrafish hE12 homologue can partially substitute for
functions of the HLH domain of Drosophila daughterless. *Mechanisms of
Development*, 46:73–85. http://doi.org/10.1016/0925-4773(94)90077-9

Yamanaka, H., & Kondo, S. (2015). Rotating pigment cells exhibit an
intrinsic chirality. *Genes to Cells*, 20:29–35.
http://doi.org/10.1111/gtc.12194

Yoshiba, S., & Hamada, H. (2014). Roles of cilia, fluid flow, and
Ca2+signaling in breaking of left-right symmetry. *Trends in Genetics*.
http://doi.org/10.1016/j.tig.2013.09.001

Zhang, M. Y., Clawson, G. A., Olivieri, N. F., Bell, L. L., Begley, C.
G., & Miller, B. A. (1997). Expression of SCL is normal in
transfusion-dependent Diamond-Blackfan anemia but other bHLH proteins
are deficient. *Blood*, 90:2068–2074. Retrieved from
http://www.ncbi.nlm.nih.gov/pubmed/9292545

Zweier, C., Peippo, M. M., Hoyer, J., Sousa, S., Bottani, A.,
Clayton-Smith, J., … Rauch, A. (2007). Haploinsufficiency of TCF4 Causes
Syndromal Mental Retardation with Intermittent Hyperventilation
(Pitt-Hopkins Syndrome). *The American Journal of Human Genetics*,
80:994–1001. http://doi.org/10.1086/515583

Chapter 8. Publications\
========================

Nakayama, M., ***Ishibashi, T.***, Ishikawa, H. O., Sato, H., Usui, T.,
Okuda, T., ... Matsuno, K. (2014). A gain-of-function screen to identify
genes that reduce lifespan in the adult of *Drosophila melanogaster*.
*BMC Genetics*, 15:46. http://doi.org/10.1186/1471-2156-15-46

Inaki, M., Hatori, R., Nakazawa, N., Okumura, T., ***Ishibashi, T.***,
Kikuta, J., ... Honda, H. (2018). Chiral cell sliding drives left-right
asymmetric organ twisting. *eLife*, 7.
http://doi.org/10.7554/eLife.32506

***Ishibashi, T.***, Hatori, R., Maeda, R., Nakamura, M., Matsuyama, Y.,
Matsuno, K. (2019) E and ID proteins regulate cell chirality and
left-right asymmetric development in *Drosophila*. *Genes to Cells*, in
press, http://doi.org/10.1111/gtc.12669

Utsunomiya, S., Sakamura, S., Sasamura, T., ***Ishibashi, T.,*** Maeda,
C., Inaki, M., Matsuno, K. submitting, Cells with broken left-right
symmetry: roles of intrinsic cell chirality in left-right asymmetric
epithelial morphogenesis. *Symmetry*

Chapter 9. Figures and Tables\
==============================

Figure 1
--------

![](media/image3.jpeg){width="2.029847987751531in"
height="4.851064085739282in"}

**Fig. 1. The *Drosophila* embryonic hindgut rotates, and it forms the
stereotypic LR asymmetric shape of the hindgut.** (A-B) Schematic
showing the LR asymmetric development of the *Drosophila* embryonic
hindgut as viewed from the dorsal side. The hindgut has an LR symmetric
shape bending dorsally (A) at stage 12, and then undergoes a
counterclockwise (broken arrow) rotation from late stage 12,
consequently bending to the right at stage 13 (B) (A’-B’) The hindgut
(orange) of wild-type embryos at stage 12 (A’) and stage 13 (B’). Dorsal
views are shown. Orange shaded area indicates the hindgut. Embryos were
stained with an anti-PY20 antibody. Scale bars are 50 μm.

Figure 2
--------

![](media/image4.png){width="6.179141513560805in"
height="3.9985028433945757in"}

**Fig. 2. *emc* is responsible for the normal LR asymmetric shape of the
embryonic hindgut.** Graph showing the frequency of LR asymmetric
defects in the embryonic hindgut. The frequencies of the hindgut showing
normal laterality (white), non-laterality (grey), inverse asymmetry
(orange), and N/A (not applicable, black) are shown as percentages. “n”
indicates the number of embryos observed. The genotypes of embryos are
indicated at the bottom. In the cases involving overexpression:
*NP2432&gt;emc::GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing
*UAS-emc::GFP* by *NP2432*; *NP2432&gt;GFP* in *emc^AP6^*, *emc^AP6^*
homozygote overexpressing *UAS-GFP*; *hand &gt; emc::GFP* in *emc^AP6^*,
*emc^AP6^* homozygote overexpressing *UAS-emc::GFP* by *hand-Gal4*;
*NP2432&gt; emc::GFP*, wild type overexpressing *UAS-emc::GFP* by
*NP2432*.

Figure 3
--------

![](media/image5.jpeg){width="6.212267060367454in"
height="3.6369772528433946in"}

**Fig. 3. The LR laterality of the hindgut is disrupted by lacking *emc*
gene.** (A-C) Dorsal view of typical embryos with the genotypes
indicated at the bottom. Typical *emc^tink^* homozygote embryo(A),
*emc^AP6^/emc^tink^* heterozygote embryo(B), *emc^AP6^* homozygote
embryo (C), *emc^AP6^* homozygote embryo overexpressing *emc::GFP*
driven by *NP2432-Gal4* (the hindgut epithelial driver) (D), and
*emc^AP6^* homozygote embryo overexpressing *emc::GFP* driven by
*hand-Gal4* (the visceral muscular driver) (E) are shown. Orange shaded
area indicates the hindgut. Embryos were stained with an anti-PY20
antibody. Scale bars are 50 μm.

Figure 4
--------

![](media/image6.jpeg){width="4.347517497812773in"
height="5.049049650043744in"}

**Fig. 4. Emc^tink^ lacks the C-terminal Gln-rich region.** (A) DNA
fragments amplified by the primers, “tink-F” and “tink-R”, corresponding
to 5’-UTR and first intron, respectively. Templates were obtained from
wild-type and *emc^tink^* homozygous embryos as indicated. The positions
of DNA size markers are indicated at the left. (B) Sequence alignment of
the wild-type *emc* (WT) and *emc^tink^* mutant (*emc^tink^*) loci. The
*emc^tink^* mutant locus has a deletion in the coding DNA sequence from
the 511st to the 531st nucleotide (shown in red). (C) Schematic showing
the predicted protein product of Emc in wild type (upper) and Emc^tink^
(bottom). Protein motifs are indicated by colored boxes: bHLH domain
(orange), Asp/Glu-rich region (cyan), and Gln-rich region (purple).
Emc^tink^ lacks the middle part of the Gln-rich region.

Figure 5
--------

![](media/image7.jpeg){width="6.100644138232721in"
height="6.507692475940507in"}

**Fig. 5. *emc* is ubiquitously expressed during the hindgut rotation.**
(A-F) *In situ* hybridization using an antisense probe (A, A’, B, B’,
and C) and sense probe (D, D’, E, E’, and F) for *emc* (blue signal).
Wild-type embryos at stage 11 (A, A’, D, and D’), stage 12 late (B, B’,
E, and E’), and stage 14 (C and F) are shown. A, B, C, D, E, and F are
dorsal views, and A’, B’, D’, and E’ are lateral views. Black dotted
lines show the outline of the hindgut. Black arrowheads indicate the
posterior midgut primordium.

Figure 6
--------

![](media/image8.jpeg){width="6.163120078740158in"
height="3.074767060367454in"}

**Fig. 6. The hindgut specification in *emc* mutant is not different
from that in wild-type.** (A) Schematic illustration showing the spatial
organization of the hindgut domains and the expression patterns of
marker genes in the domains. (B-I) Expression patterns of En (the marker
for the dorsal domain of the large intestine; B and C), Crb (the marker
for boundary cells in the large intestine; D and E), Hh (the marker for
the rectum; F and G), and Wg (the marker for the anal pad; H and I). B,
D, F, and H show the expression patterns in the wild-type embryos, and
C, E, G, and I show the expression patterns in the *emc* mutant embryos.
B, C, D, and E show the dorsal view of the embryos, and F, G, H, and I
show the lateral view of the embryos. The outlines of the hindgut are
shown by broken white lines. Scale bars are 25 μm.

Figure 7
--------

![](media/image9.png){width="3.743876859142607in"
height="3.3062325021872265in"}

**Fig. 7. Emc represses the transcriptional activity of Da.** (A-B)
Schematic diagram of the cascade composed of Emc and Da. In wild type
(A, WT), Emc inactivates Da. In the *emc* mutant (B, *emc*), Da is
derepressed, which results in a hyperactivation of Da-downstream target
genes and of the *da* transcription through its autoregulation.

Figure 8
--------

![](media/image10.png){width="6.063569553805774in" height="3.2in"}

**Fig. 8. *emc* regulates the normal LR asymmetry *via* *da*.** Graph
showing the frequency of LR asymmetric defects in the embryonic hindgut.
The frequencies of hindguts showing normal laterality (white),
non-laterality (grey), inverse asymmetry (orange), and N/A (not
applicable, black) are shown as percentages. “n” indicates the number of
embryos observed. The genotypes of embryos are indicated at the bottom.
*NP2432&gt;da* indicates embryos overexpressing *UAS-da* in the hindgut
epithelium.

Figure 9
--------

![](media/image11.jpeg){width="6.153846237970254in"
height="4.985094050743657in"}

**Fig. 9. Repression of *da* by *emc* is required for the normal LR
asymmetry of the hindgut.** (A-E) Typical phenotype of embryos with the
genotypes indicated at the bottom. The hindgut of a wild-type embryo
(WT) (A), an embryo overexpressing *UAS-da* in the hindgut epithelium
(*NP2432&gt;da*) (B), and an *emc^AP6^* homozygote (C), *da^10^*
homozygote (D), and *da^10^; emc^AP6^* double homozygote (E) embryo are
shown. Orange shaded area indicates the hindgut. Dorsal views of embryos
stained with an anti-PY20 antibody are shown. Scale bars are 25 μm.

Figure 10
---------

![](media/image12.jpg){width="6.225909886264217in"
height="2.923836395450569in"}

**Fig. 10. The expression level of *da* is upregulated in *emc* mutant
embryos.** Bar graph showing the relative expression level of *da* in
wild-type (WT, white) and the *emc^AP6^* homozygote (*emc^AP6^*, grey).
The expression level relative to the wild-type level is shown. Standard
error is shown by black lines. “n” and “N” represent the numbers of
total samples and the biological replicates, respectively. The results
of a Welch's 2-sample t-test with a 5% significance level are shown.
Asterisk indicates p &lt; 0.05.

Figure 11
---------

![](media/image13.png){width="6.053846237970253in"
height="5.813788276465441in"}

**Fig. 11. *Myo31DF* overexpression rescued the LR asymmetric defects
observed in *emc* mutants.** Graph showing the frequency of LR asymmetry
defects in the embryonic hindgut. The frequencies of hindguts showing
normal laterality (white), non-laterality (grey), inverse asymmetry
(orange), and N/A (not applicable, black) are shown as percentages. “n”
indicates the number of embryos observed. The genotypes of embryos are
indicated at the bottom. In the cases involving overexpression:
*NP2432&gt;Myo31DF::RFP* in *Myo31DF^K2^*, *Myo31DF^K2^* homozygote
overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432&gt;Myo31DF::RFP*
in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP*
by *NP2432*; *NP2432&gt;da*, wild type overexpressing *da* by *NP2432*;
*NP2432&gt;da&gt;Myo31DF::RFP*, wild-type overexpressing *UAS-da* and
*UAS-Myo31DF::RFP* by *NP2432*; *NP2432&gt;da&gt;GFP*, wild type
overexpressing *UAS-da* and *UAS-GFP* by *NP2432.*

Figure 12
---------

![](media/image14.jpeg){width="6.170468066491688in"
height="3.9230774278215224in"}

**Fig. 12. Disrupted LR asymmetry of the hindgut in *emc* mutants are
rescued by overexpression of *Myo31DF*.** (A-G) Typical phenotype of
embryos with the genotypes indicated at the bottom. The hindgut of wild
type (WT) (A), *Myo31DF^K2^* homozygote (B), *emc^AP6^* homozygote (C),
*Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP*
(*NP2432&gt;Myo31DF::RFP* in *Myo31DF^K2^*) (D), *emc^AP6^* homozygote
overexpressing *UAS-Myo31DF::RFP* (*NP2432&gt;Myo31DF::RFP* in
*emc^AP6^*) (E), an embryo overexpressing *UAS*-*da* (*NP2432&gt;da*)
(F), and an embryo overexpressing *UAS-da* and *UAS-Myo31DF::RFP*
(*NP2432&gt;da&gt;Myo31DF::RFP*) (G) are shown. Orange shaded area
indicates the hindgut. Dorsal views of embryos stained with an anti-PY20
antibody are shown. Scale bars are 25 μm.

Figure 13
---------

![](media/image15.jpeg){width="4.943173665791776in"
height="2.6458234908136484in"}

**Fig. 13. The expression level of *Myo31DF* is not significantly
changed in *emc* mutants.** Bar graph showing the relative expression
level of *Myo31DF* in wild-type (WT, white) and the *emc^AP6^*
homozygote (*emc^AP6^*, grey). The expression level relative to the
wild-type level is shown. Standard error is shown by black lines. “n”
and “N” represent the numbers of total samples and the biological
replicates, respectively. For the statistical analysis, a Welch's
2-sample t-test with a 5% significance level was used.

Figure 14
---------

![](media/image16.jpeg){width="5.130441819772528in"
height="5.827104111986002in"}

**Fig. 14. Localization of Myo31DF protein is not changed in *emc*
lacking cells.** (A-C) Mosaic hindgut epithelium composed of *emc^AP6^*
mutant cells and the same mutant cells overexpressing *UAS-emc::GFP*,
stained with Myo31DF (A) and anti-GFP (B). C is a merged image of A
(shown in magenta) and B (shown in green). A’, B’, and C’ are
high-magnification images of the regions indicated by white boxes in A,
B, and C, respectively. In A’, asterisks indicate *emc* mutant cells
overexpressing *UAS-emc::GFP*. Scale bar indicates 10 μm. (D) Graph
showing the normalized signal intensity of anti-Myo31DF antibody
staining. In the hindgut stained with anti-Myo31DF antibody, the
normalized mean intensity of the Myo31DF signal along the apical,
lateral and basal cell boundaries in wild-type cells (white) and
*emc^AP6^* homozygous cells (grey) was calculated. “n” represents the
numbers of cell boundaries analyzed. Standard error is shown by black
lines. For the statistical analysis, a Welch's 2-sample t-test with a 5%
significance level was used.

Figure 15
---------

![](media/image17.jpeg){width="5.275624453193351in"
height="2.615793963254593in"}

**Fig. 15. The expression level of *Myo61F* is not significantly changed
in *emc* mutants.** Bar graph showing the relative expression level of
*Myo61F* in wild-type (WT, white) and the *emc^AP6^* homozygote
(*emc^AP6^*, grey). The expression level relative to the wild-type level
is shown. Standard error is shown by black lines. “n” and “N” represent
the numbers of total samples and the biological replicates,
respectively. For the statistical analysis, a Welch's 2-sample t-test
with a 5% significance level was used.

Figure 16
---------

![](media/image18.jpeg){width="6.02830927384077in"
height="4.794326334208224in"}

**Fig. 16. The expression level and localization of DE-Cad are not
changed in *emc* mutants.** (A-B) Western blot analysis of the DE-Cad
protein (A) and quantification of the band intensity (B). Total proteins
extracted from wild type (WT) and *emc^AP6^* embryos at approximately
stage 12-14 were analyzed. The positions of molecular-weight markers are
indicated at right. α-Tubulin (α-Tub) was detected as an internal
control for protein loading. In B’, the ratio of DE-Cad intensity to
that of α-Tub was determined, and the relative value of DE-Cad in
*emc^AP6^* relative to that in wild type was determined. (C-D) The
embryonic hindgut of wild type (WT) (C) and the *emc^AP6^* homozygote
(D) stained with anti-DE-Cad (magenta in C and D) and PY20 (green in C’
and D’) antibodies. C’’ and D’’ are merged images of C and C’ and D and
D’, respectively. Scale bars are 10 μm. (E) Graph showing the normalized
signal intensity of anti-DE-Cad antibody staining. In the hindgut
double-stained with anti-DE-Cad and PY20 antibodies, the ratio of the
DE-Cad signal along the cell boundaries to that of the PY20 staining was
calculated. “N” and “n” represent the numbers of embryos and cell
boundaries analyzed, respectively. Standard error is shown by black
lines. For the statistical analysis, a Welch's 2-sample t-test with a 5%
significance level was used.

Figure 17
---------

![](media/image19.png){width="3.127658573928259in"
height="3.796198600174978in"}

**Fig. 17. Hindgut epithelial cells show the cell chirality before the
hindgut rotates.** (A-B) Schematic showing the relationship between the
cell chirality of the hindgut epithelial cells and hindgut rotation
(arrow) (A). (A)Before rotation, the apical cell boundaries (black lines
in the square) of the hindgut epithelial cells show LR-asymmetry, with
cell boundaries that tend to slant to the left. (B) After rotation, the
LR-asymmetry of the cells (black lines in the square) disappears.

Figure 18
---------

![](media/image20.png){width="5.507054899387577in"
height="3.780142169728784in"}

**Fig. 18. Definition of the chirality index.** (A-B) Schematic
summarizing the procedure used to calculate the chirality index. The
apical cell boundaries are visualized by anti-DE-Cad staining in the
wild-type hindgut. The angle between each boundary and the
anterior-posterior (AP) axis of the hindgut is determined in a
double-blind manner (A). LR represents the left-right axis. Each
boundary is classified as a right-tilted boundary (orange) or
left-tilted boundary (turquoise) (B). The numbers of right-tilted (N~R~)
and left-tilted (N~L~) boundaries were counted, and the chirality index
was calculated using the formula shown at the bottom of B.

Figure 19
---------

![](media/image21.jpeg){width="4.095110454943132in"
height="7.9078007436570426in"}

**Fig. 19. Chirality index is disrupted in *emc* mutants, which is
rescued by *Myo31DF* overexpression.** (A) Strip plots showing the
chirality index of embryos with the genotypes indicated at the bottom.
The mean values (orange line) and standard errors (black dotted line)
are presented. The grey dotted line indicates the 0 value of the
chirality index. “n” indicates the number of total boundaries analyzed,
and “N” indicates the number of total embryos analyzed. In the cases
involving overexpression: *NP2432&gt;Myo31DF::RFP* in *Myo31DF^K2^*,
*Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*;
*NP2432&gt;emc::GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing
*UAS-emc::GFP* by *NP2432*; *NP2432&gt;Myo31DF::RFP* in *emc^AP6^*,
*emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*;
*NP2432&gt;da*, wild-type overexpressing *UAS-da* by *NP2432*;
*NP2432&gt;da&gt;Myo31DF::RFP*, wild-type overexpressing *UAS-da* and
*UAS-Myo31DF::RFP* by *NP2432*. (B) Matrix showing the statistical
significances of the results shown in A. For this analysis,
Tukey-Kramer’s multiple comparison test was done between all of the
combinations shown in A. The color of each square represents statistical
significance between the genotypes presented at the left and at the
bottom side of the matrix. Yellow, orange, and red squares indicate p
&lt; 0.05, p &lt; 0.01, and p &lt; 0.001, respectively. White square
indicates p ≥ 0.05 (no significance).

Figure 20
---------

![](media/image22.jpeg){width="6.049280402449694in"
height="3.2411351706036746in"}

**Fig. 20. *emc* and *da* are involved in the normal cell chirality
formation** (A-H) Typical examples of the apical cell boundaries in the
hindgut epithelium of embryos with the indicated genotypes. Wild-type
(WT) (A), *Myo31DF^K2^* homozygote (B), *emc^AP6^* homozygote (C),
*emc^AP6^* homozygote overexpressing *UAS-emc::GFP* (D), wild-type
embryo overexpressing *UAS-da* (E), *Myo31DF^K2^* homozygote
overexpressing *UAS-Myo31DF::RFP* (F), *emc^AP6^* homozygote
overexpressing *UAS-Myo31DF::RFP* (G), and wild-type embryo
double-overexpressing *UAS-da* and *UAS-Myo31DF::RFP* (H) are shown.
Overexpression was driven by *NP2432* in D, E, F, G, and H. The apical
cell boundaries were visualized by anti-*D*E-Cad antibody staining.
Scale bars are 2.5 μm.

Figure 21
---------

![](media/image23.png){width="5.921563867016623in"
height="1.6658945756780403in"}

**Fig. 21. A model summarizing my thesis; Emc, Da, and Myo31DF are
involved in the normal cell chirality formation.** A model summarizing
the revealed cascade involving Emc, Da, and Myo31DF in the cell
chirality formation and counterclockwise rotation of the hindgut. The
repression of Da by Emc is required for the formation of the normal cell
chirality. Our group suggested that the normal cell chirality induced
the wild-type rotation of the hindgut. Myo31DF regulates the cell
chirality under the control of or parallel to Emc/Da.

Figure 22
---------

![](media/image24.png){width="6.056136264216973in"
height="7.798570647419073in"}

**Fig. 22. Chirality index sigmoidally correlates with the probability
of the wild-type LR asymmetry of the hindgut in various embryos.**
Scatter plot showing the relationship between in chirality index and the
probability of the normal LR laterality in the hindgut of embryos with
the genotypes indicated at the bottom. For the probability data, the
phenotype frequencies of the LR asymmetry in the hindgut that calculated
in Fig. 2, 8, and 11 are used. For the chirality index data, the same
values calculated in Fig. 19 are used. Standard errors (horizontal
lines) are presented according to Fig. 19. The grey solid line and
broken line indicate the linear fitting model and Hill equation fitting
model, respectively. Fitting parameters are shown in Table 3. The
numbers of embryos for the chirality index and for the probability of
the LR asymmetry are shown in the bottom panel as *N~C.I.~* and
*n~Prob.~*, respectively. Correlation coefficient, coefficient of
determination of the fitted linear model, and coefficient of
determination of the fitted Hill equation model are shown at the upper
right corner as R, Linear R^2^, and Hill R^2^, respectively.

Figure 23
---------

![../OneDrive%20-%20大阪大学/emc-Figs-pre/PhD\_Thesis/figs/23\_new/23.png](media/image25.png){width="6.228412073490814in"
height="4.495326990376203in"}

**Fig. 23. Live chirality index sigmoidally correlates with the LR
laterality of the individual *emc^AP6^* mutant embryos.** Scatter plot
showing the relationship between in live chirality index and the LR
laterality of the hindgut in individual *emc^AP6^* mutant embryos. The
grey solid line and broken line indicate the linear fitting model and
Hill equation fitting model, respectively. Fitting parameters are shown
in Table 3. Correlation coefficient, coefficient of determination of the
fitted linear model, and coefficient of determination of the fitted Hill
equation model are shown at the upper right corner as R, Linear R^2^,
and Hill R^2^, respectively.

Figure 24
---------

![](media/image26.jpeg){width="4.102457349081365in"
height="7.921984908136483in"}

**Fig. 24. The apical size is expanded in *emc*, but the expansion is
not relevant to the cell chirality or LR asymmetry of the hindgut.** (A)
Strip plots showing the size of the apical cell surface in the hindgut
epithelium of embryos with the genotypes indicated at the bottom. The
mean value (orange line) and standard error (black dotted line) are
presented. “n” and “N” indicate the number of total cells and total
embryos analyzed, respectively. In the cases involving overexpression:
*NP2432&gt;Myo31DF::RFP* in *Myo31DF^K2^*, *Myo31DF^K2^* homozygote
overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432&gt;emc::GFP* in
*emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-emc::GFP* by
*NP2432*; *NP2432&gt;Myo31DF::RFP* in *emc^AP6^*, *emc^AP6^* homozygote
overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432&gt;da*, wild-type
overexpressing *UAS-da* by *NP2432*; *NP2432&gt;da&gt;Myo31DF::RFP*,
wild-type overexpressing *UAS-da* and *UAS-Myo31DF::RFP* by *NP2432*.
(B) Matrix showing the statistical significances of the results shown in
A. For this analysis, Tukey-Kramer’s multiple comparison test was done
between all of the combinations shown in A. The color of each square
represents statistical significance between the genotypes presented at
the left and at the bottom side of the matrix. Yellow, orange, and red
squares indicate p &lt; 0.05, p &lt; 0.01, and p &lt; 0.001,
respectively. White square indicates p ≥ 0.05 (no significance).

Figure 25
---------

![](media/image27.jpeg){width="4.8936165791776025in"
height="5.336332020997375in"}

**Fig. 25. F-actin localization is not changed in *emc* mutant cells.**
(A-C) Mosaic hindgut epithelium composed of *emc^AP6^* mutant cells and
the same mutant cells overexpressing *UAS-emc::GFP*, stained with
phalloidin (A) and anti-GFP (B). C is a merged image of A (shown in
green) and B (shown in magenta). A’, B’, and C’ are high-magnification
images of the regions indicated by white boxes in A, B, and C,
respectively. In A’, asterisks indicate *emc* mutant cells
overexpressing *UAS-emc::GFP*, and the arrowhead shows the apical region
of the hindgut epithelial cells. Scale bar indicates 20 μm (D) Graph
showing the normalized signal intensity of phalloidin staining. In the
hindgut stained with rhodamine phalloidin, the normalized mean intensity
of the rhodamine signal along the apical, lateral and basal cell
boundaries in wild-type cells (white) and *emc^AP6^* homozygous cells
(grey) was calculated. “n” represents the numbers of cell boundaries
analyzed. Standard error is shown by black lines. For the statistical
analysis, a Welch's 2-sample t-test with a 5% significance level was
used.

Figure 26
---------

![](media/image28.jpg){width="5.857726377952756in"
height="3.8368799212598423in"}

**Fig. 26. The number of the hindgut epithelial cells is not changed in
*emc* mutants.** (A-B) Typical examples of the hindgut stained with
anti-Lamin C antibody. Wild-type (A) and *emc^AP6^* homozygous mutant
(B) were shown. White dotted lines show the outline of the hindgut.
Scale bar indicates 25 μm. (C) Bar graph showing the cell number in
wild-type (left) and *emc^AP6^* mutant (right). Standard error is shown
by black lines. “N” indicates the number of total embryos analyzed. For
the statistical analysis, a Welch's 2-sample t-test with a 5%
significance level was used.

Figure 27
---------

![../OneDrive%20-%20大阪大学/emc-Figs-pre/PhD\_Thesis/figs/27/27.png](media/image29.png){width="4.407950568678915in"
height="7.774107611548557in"}

**Fig. 27. A model where the cell chirality determines the rotational
direction of the hindgut.** (A) Schematic showing the possible model in
which the cell chirality determines the rotational direction of the
hindgut. There are local minimums of the unstability with rotational
direction to -90º (wild-type direction), 0º (non-lateral), and +90º
(inverse direction), and this unstability is changed depending on the
cell chirality. Perturbations in individual embryos fluctuate, and it
stochastically determines the rotational direction of the hindgut. (B)
In wild-type, the enough dextral cell chirality is generated, which
induces - 90º-biased unstability. Therefore, the hindgut rotates to
wild-type direction at 100% probability. (B’) In *emc* mutants, the cell
chirality disappears and there are three local minimums of the
unstability with rotational direction. In this condition, the
fluctuations of the cell chirality in individual embryos stochastically
determine the rotational direction of the hindgut.

Figure 28
---------

![](media/image30.png){width="5.9006944444444445in"
height="2.60040791776028in"}

**Fig. 28. A model summarizing the novel role of E/Id regulatory system
in the cell shape and cell size regulation revealed by this thesis.** In
this thesis, I revealed that *emc* and *da* is involved in the
regulation of cell shape, such as the cell chirality and the apical
size. It was suggested that the cell shape regulated by *emc* is
required for the proper control of organ shape.

Table 1
-------

  ----------------------------------------------------------------------------------------------------------------------------------------------
  **Gene name**          **Molecular Function**                                    **Reference**
  ---------------------- --------------------------------------------------------- -------------------------------------------------------------
  *extra macrochaetae*   Transcription corepressor of bHLH transcription factors   (Garrell & Modolell, 1990)

  *Notch*                Transmembrane signaling receptor                          (Kidd et al., 1986; Rice & Liu, 2015; Wharton et al., 1985)
                                                                                   
                         Notch signaling component                                 

  *Antennapedia*         Homeobox transcription factor                             (Laughon et al., 1986; McGinnis et al., 1984)

  *Deformed*             Homeobox transcription factor                             (Regulski et al., 1985)

  *engrailed*            Homeobox transcription factor                             (Kassis et al., 1986)

  *cut*                  Homeobox transcription factor                             (Blochlinger et al., 1988)

  *bicoid*               Homeobox transcription factor                             (Berleth et al., 1988)

  *daughterless*         bHLH transcription factor                                 (Cronmiller et al., 1988)

  *twist*                bHLH transcription factor                                 (Thisse et al., 1988)

  *single-minded*        bHLH transcription factor                                 (Crews et al., 1988)

  *taiman*               bHLH transcription factor                                 (Bai et al., 2000)

  *hairy*                bHLH transcription corepressor                            (Wainwright & Ish-Horowicz, 1992)

  *hunchback*            Zinc finger transcription factor                          (Tautz et al., 1987)

  *dorsal*               NF-κB family transcription factor                         (Steward, 1987)
                                                                                   
                         Toll signaling component                                  
  ----------------------------------------------------------------------------------------------------------------------------------------------

**Table 1. Various *Drosophila* proteins have Gln-rich region.** Table
showing the genes including *opa* repeat in *Drosophila*. The
*Drosophila* gene that has Gln-rich region, the molecular function and
the paper where the protein structure of the genes is reported are shown
in the first, the second, and the third column, respectively.

Table 2
-------

  *emc* expression in the hindgut   stage 11   stage 12   stage 14
  --------------------------------- ---------- ---------- ----------
  +                                 11         4          0
  +/-                               1          l3         7
  -                                 0          0          14
  Total                             12         17         21

**Table 2. *emc* expresses in the hindgut during the hindgut rotation.**
The expression of *emc* was detected by *in situ* hybridization using an
antisense probe for *emc* (see Figure 5). The numbers of embryos showing
*emc* signals in the hindgut primordium and hindgut are shown. +, strong
signal; +/-, weak signal; -, no signal.

Table 3
-------

  -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
      Fig. 22                                                                                                     Fig. 23
                                                                                                                  
      (Chirality index v.s. Probability of the normal LR asymmetry)                                               (Live chirality index v.s. Rotational direction of the hindgut in *emc^AP6^*)
  --- ----------------------------------------------------------------------------------------------------------- ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ----------------------------------------------------------------------------------------------------------- ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
      Linear model                                                                                                Hill curve model                                                                                                                                                             Linear model                                                                                                Hill curve model
                                                                                                                                                                                                                                                                                                                                                                                                           
      ![../Downloads/texclip20190116162807.png](media/image1.png){width="1.05in" height="0.2520833333333333in"}   ![../OneDrive%20-%20大阪大学/emc-Figs-pre/PhD\_Thesis/figs/22-new/texclip20190116163452.png](media/image2.png){width="1.2429910323709537in" height="0.3318153980752406in"}   ![../Downloads/texclip20190116162807.png](media/image1.png){width="1.05in" height="0.2520833333333333in"}   ![../OneDrive%20-%20大阪大学/emc-Figs-pre/PhD\_Thesis/figs/22-new/texclip20190116163452.png](media/image2.png){width="1.2429910323709537in" height="0.3318153980752406in"}

  α   -2.65542403                                                                                                 0.80126171                                                                                                                                                                   -2.67589192                                                                                                 0.85349606

  β   0.67360137                                                                                                  0.051899117                                                                                                                                                                  0.58362428                                                                                                  0.044309219

  n                                                                                                               20.8339558                                                                                                                                                                                                                                                                               21.843073
  -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

**Table 3. Fitting parameters in Fig. 22 and 23.** The fitting equations
and parameters used in Fig. 22 and Fig. 23 are shown.

Chapter 10. Acknowledgements\
=============================

I would like to express my deepest profound gratitude to my supervisor,
Professor Kenji Matsuno. He provided me the precious research
opportunity in his laboratory. He always gave me the suggestive and
constructive feedback that have strongly encouraged me. As time goes on,
I realize more and more clearly the huge impact that he has had on my
academic career. I cannot even imagine a better supervisor than him.

I would like to express my deep sincere appreciations to Dr. Tomoko
Yamakawa and Dr. Mikiko Inaki for the considerable encouragement and
invaluable discussions. I will not forget their warm words to encourage
me. Their presence was quite important to study in Matusno lab for me.

I also want to express my deep gratitude to Hiroyuki Iida and Kentaro
Yamamoto who have shared moments of deep anxiety but also of great
excitement. They have taken a lot of time to discuss that stimulates and
enriches my works.

I am tremendously grateful to Professor Koichi Fujimoto, Dr. Ryo Hatori,
Dr. Mitsutoshi Nakamura, Dr. Takeshi Sasamura, Dr. Satoshi Kuwana, Dr.
Yoshiko Aihara, Dr. Tomonori Deguchi, Dr. Atsuo Iida, Dr. Takeshi Onuma,
Dr. Hiroko Udaka, Dr. Shizue Ohsawa, Dr. Naoki Okamoto, Dr. Shinya
Yamamoto, and Matsuno lab members for the fruitful discussions. I am
also thankful to Momoko Inatomi, Dongsun Shin, Puspa Das, Yi-Ting Lai,
Sosuke Utsunomiya, and all my Ph.D. colleagues who shared the great
times.

I deeply acknowledge the members of my Ph.D. committee, Professor Hiroki
Nishida and Professor Hiroshi Sasaki, for their time and valuable
feedback on the preliminary thesis.

I am very grateful to Dr. Takehiko Usui who gave me the great expertise
on the molecular works when I was a technical staff at Shinohara lab,
Institute for Protein Research. The techniques he provided me always
help me to perform experiments. I also thank to Dr. Naoto Ueno, Dr.
Yusuke Hara, Dr. Yasushi Hiromi, and Dr. Takashi Hayashi. Without them,
I did not decide to belong to Matsuno lab.

I would like to thank Dr. Nam-Sung Moon, Dr. Stéphane Noselli, and Dr.
Hiroyuki O Ishikawa for fly stocks. I also thank the Bloomington
Drosophila Stock Center (Indiana University), the Drosophila Genetic
Resource Center (Indiana University), and the Kyoto Stock Center (Kyoto
Institute of Technology) for fly stocks, and the Developmental Studies
Hybridoma Bank (University of Iowa) for antibodies.

I sincerely acknowledge Yoshimura Foundation and Japan Society for the
Promotion of Science for the financial supports during my M.S. and Ph.D.
program.

Finally, I am deeply thankful to my family, especially to my brother
Naoki Ishibashi and my mother Noriko Ishibashi, for their constant
support and sacrifices. They gave me the freedom to study at the
university. The last word goes for Nariyasu Ishibashi, my late father. I
hope to inherit his talents for languages and philosophy. I dedicate
this thesis to him.
