# Figures and Tables

![](../figs/01/1.jpg){height=17cm}  
**Fig. 1. The *Drosophila* embryonic hindgut rotates, and it forms the stereotypic LR asymmetric shape of the hindgut.**
(A-B) Schematic showing the LR asymmetric development of the *Drosophila* embryonic hindgut as viewed from the dorsal side.
The hindgut has an LR symmetric shape bending dorsally (A) at stage 12, and then undergoes a counterclockwise (broken arrow) rotation from late stage 12, consequently bending to the right at stage 13 (B) (A'-B') The hindgut (orange) of wild-type embryos at stage 12 (A') and stage 13 (B').
Dorsal views are shown.
Orange shaded area indicates the hindgut.
Embryos were stained with an anti-PY20 antibody.
Scale bars are 50 μm.

\clearpage

![](../figs/02/2.png)  
**Fig. 2. *emc* is responsible for the normal LR asymmetric shape of the embryonic hindgut.**
Graph showing the frequency of LR asymmetric defects in the embryonic hindgut.
The frequencies of the hindgut showing normal laterality (white), non-laterality (grey), inverse asymmetry (orange), and N/A (not applicable; LR laterality was unscorable due to the hindgut deformation) (black) are shown as percentages.
"n"indicates the number of embryos observed.
The genotypes of embryos are indicated at the bottom.
In the cases involving overexpression: *NP2432>emc::GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-emc::GFP* by *NP2432*; *NP2432>GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-GFP*; *hand>emc::GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-emc::GFP* by *hand-Gal4*; *NP2432>emc::GFP*, wild type overexpressing *UAS-emc::GFP* by *NP2432*.

\clearpage

![](../figs/03/3.jpg)  
**Fig. 3. The LR laterality of the hindgut is disrupted by lacking *emc* gene.**
(A-C) Dorsal view of typical embryos with the genotypes indicated at the bottom.
Typical *emc^tink^* homozygote embryo(A), *emc^AP6^/emc^tink^* heterozygote embryo(B), *emc^AP6^* homozygote embryo (C), *emc^AP6^* homozygote embryo overexpressing *emc::GFP* driven by *NP2432-Gal4* (the hindgut epithelial driver) (D), and *emc^AP6^* homozygote embryo overexpressing *emc::GFP* driven by *hand-Gal4* (the visceral muscular driver) (E) are shown.
Orange shaded area indicates the hindgut.
Embryos were stained with an anti-PY20 antibody.
Scale bars are 50 μm.

\clearpage

![](../figs/04/4.jpg){height=120mm}  
**Fig. 4. Emc^tink^ lacks the C-terminal Gln-rich region.**
(A) DNA fragments amplified by the primers, "tink-F" and "tink-R", corresponding to 5'-UTR and first intron, respectively.
Templates were obtained from wild-type and *emc^tink^* homozygous embryos as indicated.
The positions of DNA size markers are indicated at the left.
(B) Sequence alignment of the wild-type *emc* (WT) and *emc^tink^* mutant (*emc^tink^*) loci.
The *emc^tink^* mutant locus has a deletion in the coding DNA sequence from the 511st to the 531st nucleotide (shown in red).
(C) Schematic showing the predicted protein product of Emc in wild type (upper) and Emc^tink^ (bottom).
Protein motifs are indicated by colored boxes: bHLH domain (orange), Asp/Glu-rich region (cyan), and Gln-rich region (purple).
Emc^tink^ lacks the middle part of the Gln-rich region.

\clearpage

![](../figs/05/5.jpg)  
**Fig. 5. *emc* is ubiquitously expressed during the hindgut rotation.**
(A-F) *In situ* hybridization using an antisense probe (A, A', B, B', C, C', D, D', E, and E') and sense probe (F, G, H, I, and J) for *emc* (blue signal).
Wild-type embryos at stage 11 (A, A', B, B', F, and G), stage 12 late (C, C', D, D', H, and I), and stage 14 (E, E', and J) are shown.
A, C, E, F, H and J are dorsal views, and B, D, G, and I are lateral views.
A', B', C', D', and E' are high-magnification images of the regions indicated by white squares in A, B, C, D, and E, respectively.
Black dotted lines show the outline of the hindgut.
Black arrowheads indicate the posterior midgut primordium.

\clearpage

![](../figs/06/6.jpg)  
**Fig. 6. The hindgut specification in *emc* mutant is not different from that in wild-type.**
(A) Schematic illustration showing the spatial organization of the hindgut domains and the expression patterns of marker genes in the domains.
(B-I) Expression patterns of En (the marker for the dorsal domain of the large intestine; B and C), Crb (the marker for boundary cells in the large intestine; D and E), Hh (the marker for the rectum; F and G), and Wg (the marker for the anal pad; H and I).
B, D, F, and H show the expression patterns in the wild-type embryos, and C, E, G, and I show the expression patterns in the *emc* mutant embryos.
B, C, D, and E show the dorsal view of the embryos, and F, G, H, and I show the lateral view of the embryos.
The outlines of the hindgut are shown by broken white lines.
Scale bars are 25 μm.

\clearpage

![](../figs/07/7.png)  
**Fig. 7. Emc represses the transcriptional activity of Da.**
(A-B) Schematic diagram of the cascade composed of Emc and Da.
In wild type (A, WT), Emc inactivates Da.
In the *emc* mutant (B, *emc*), Da is derepressed, which results in a hyperactivation of Da-downstream target genes and of the *da* transcription through its autoregulation.

\clearpage

![](../figs/08/8.png)
**Fig. 8. *emc* regulates the normal LR asymmetry *via* *da*.**
Graph showing the frequency of LR asymmetric defects in the embryonic hindgut.
The frequencies of hindguts showing normal laterality (white),
non-laterality (grey), inverse asymmetry (orange), and N/A (not
applicable; LR laterality was unscorable due to the hindgut deformation) (black) are shown as percentages.
"n" indicates the number of
embryos observed.
The genotypes of embryos are indicated at the bottom.
*NP2432>da* indicates embryos overexpressing *UAS-da* in the hindgut
epithelium.

\clearpage

![](../figs/09/9.jpg)  
**Fig. 9. Repression of *da* by *emc* is required for the normal LR asymmetry of the hindgut.**
(A-E) Typical phenotype of embryos with the
genotypes indicated at the bottom.
The hindgut of a wild-type embryo
(WT) (A), an embryo overexpressing *UAS-da* in the hindgut epithelium
(*NP2432>da*) (B), and an *emc^AP6^* homozygote (C), *da^10^*
homozygote (D), and *da^10^; emc^AP6^* double homozygote (E) embryo are
shown.
Orange shaded area indicates the hindgut.
Dorsal views of embryos
stained with an anti-PY20 antibody are shown.
Scale bars are 25 μm.

\clearpage

![](../figs/10/10.jpg){height=100mm}  
**Fig. 10. The expression level of *da* is upregulated in *emc* mutant embryos.**
Bar graph showing the relative expression level of *da* in wild-type (WT, white) and the *emc^AP6^* homozygote (*emc^AP6^*, grey).
The expression level relative to the wild-type level is shown.
Standard error is shown by black lines.
"n" and "N" represent the numbers of total samples and the biological replicates, respectively.
The results of a Welch's 2-sample t-test with a 5% significance level are shown.
Asterisk indicates p < 0.05.

\clearpage

![](../figs/11/11.png)  
**Fig. 11. *Myo31DF* overexpression rescued the LR asymmetric defects observed in *emc* mutants.**
Graph showing the frequency of LR asymmetry defects in the embryonic hindgut.
The frequencies of hindguts showing normal laterality (white), non-laterality (grey), inverse asymmetry (orange), and N/A (not applicable; LR laterality was unscorable due to the hindgut deformation) (black) are shown as percentages.
"n"indicates the number of embryos observed.
The genotypes of embryos are indicated at the bottom.
In the cases involving overexpression: *NP2432>Myo31DF::RFP* in *Myo31DF^K2^*, *Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>Myo31DF::RFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>da*, wild type overexpressing *da* by *NP2432*; *NP2432>da>Myo31DF::RFP*, wild-type overexpressing *UAS-da* and *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>da>GFP*, wild type overexpressing *UAS-da* and *UAS-GFP* by *NP2432.*

\clearpage

![](../figs/12/12.jpg)  
**Fig. 12. Disrupted LR asymmetry of the hindgut in *emc* mutants are rescued by overexpression of *Myo31DF*.**
(A-G) Typical phenotype of embryos with the genotypes indicated at the bottom.
The hindgut of wild type (WT) (A), *Myo31DF^K2^* homozygote (B), *emc^AP6^* homozygote (C), *Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP* (*NP2432>Myo31DF::RFP* in *Myo31DF^K2^*) (D), *emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP* (*NP2432>Myo31DF::RFP* in *emc^AP6^*) (E), an embryo overexpressing *UAS*-*da* (*NP2432>da*) (F), and an embryo overexpressing *UAS-da* and *UAS-Myo31DF::RFP* (*NP2432>da>Myo31DF::RFP*) (G) are shown.
Orange shaded area indicates the hindgut.
Dorsal views of embryos stained with an anti-PY20 antibody are shown.
Scale bars are 25 μm.

\clearpage

![](../figs/13/13.jpg)  
**Fig. 13. The expression level of *Myo31DF* is not significantly changed in *emc* mutants.**
Bar graph showing the relative expression level of *Myo31DF* in wild-type (WT, white) and the *emc^AP6^* homozygote (*emc^AP6^*, grey).
The expression level relative to the wild-type level is shown.
Standard error is shown by black lines.
"n"and "N" represent the numbers of total samples and the biological replicates, respectively.
For the statistical analysis, a Welch's 2-sample t-test with a 5% significance level was used.

\clearpage

![](../figs/14/14.jpg)  
**Fig. 14. Localization of Myo31DF protein is not changed in *emc* lacking cells.**
(A-C) Mosaic hindgut epithelium composed of *emc^AP6^* mutant cells and the same mutant cells overexpressing *UAS-emc::GFP*, stained with Myo31DF (A) and anti-GFP (B).
C is a merged image of A (shown in magenta) and B (shown in green).
A', B', and C' are high-magnification images of the regions indicated by white boxes in A, B, and C, respectively.
In A', asterisks indicate *emc* mutant cells overexpressing *UAS-emc::GFP*.
Scale bar indicates 10 μm.
(D) Graph showing the normalized signal intensity of anti-Myo31DF antibody staining.
In the hindgut stained with anti-Myo31DF antibody, the normalized mean intensity of the Myo31DF signal along the apical, lateral and basal cell boundaries in wild-type cells (white) and *emc^AP6^* homozygous cells (grey) was calculated.
"n" represents the numbers of cell boundaries analyzed.
Standard error is shown by black lines.
For the statistical analysis, a Welch's 2-sample t-test with a 5% significance level was used.

\clearpage

![](../figs/15/15.jpg)  
**Fig. 15. The expression level of *Myo61F* is not significantly changed in *emc* mutants.**
Bar graph showing the relative expression level of *Myo61F* in wild-type (WT, white) and the *emc^AP6^* homozygote (*emc^AP6^*, grey).
The expression level relative to the wild-type level is shown.
Standard error is shown by black lines.
"n" and "N" represent the numbers of total samples and the biological replicates, respectively.
For the statistical analysis, a Welch's 2-sample t-test with a 5% significance level was used.

\clearpage

![](../figs/16/16.jpg)  
**Fig. 16. The expression level and localization of DE-Cad are not changed in *emc* mutants.**
(A-B) Western blot analysis of the DE-Cad protein (A) and quantification of the band intensity (B).
Total proteins extracted from wild type (WT) and *emc^AP6^* embryos at approximately stage 12-14 were analyzed.
The positions of molecular-weight markers are indicated at right.
α-Tubulin (α-Tub) was detected as an internal control for protein loading.
In B', the ratio of DE-Cad intensity to that of α-Tub was determined, and the relative value of DE-Cad in *emc^AP6^* relative to that in wild type was determined.
(C-D) The embryonic hindgut of wild type (WT) (C) and the *emc^AP6^* homozygote (D) stained with anti-DE-Cad (magenta in C and D) and PY20 (green in C'and D') antibodies.
C'' and D'' are merged images of C and C' and D and D', respectively.
Scale bars are 10 μm.
(E) Graph showing the normalized signal intensity of anti-DE-Cad antibody staining.
In the hindgut double-stained with anti-DE-Cad and PY20 antibodies, the ratio of the DE-Cad signal along the cell boundaries to that of the PY20 staining was calculated.
"N" and "n" represent the numbers of embryos and cell boundaries analyzed, respectively.
Standard error is shown by black lines.
For the statistical analysis, a Welch's 2-sample t-test with a 5% significance level was used.

\clearpage

![](../figs/17/17.png){height=100mm}  
**Fig. 17. Hindgut epithelial cells show the cell chirality before the hindgut rotates.**
(A-B) Schematic showing the relationship between the cell chirality of the hindgut epithelial cells and hindgut rotation (arrow) (A).
(A)Before rotation, the apical cell boundaries (black lines in the square) of the hindgut epithelial cells show LR-asymmetry, with cell boundaries that tend to slant to the left.
(B) After rotation, the LR-asymmetry of the cells (black lines in the square) disappears.

\clearpage

![](../figs/18/18.png)  
**Fig. 18. Definition of the chirality index.**
(A-B) Schematic summarizing the procedure used to calculate the chirality index.
The apical cell boundaries are visualized by anti-DE-Cad staining in the wild-type hindgut.
The angle between each boundary and the anterior-posterior (AP) axis of the hindgut is determined in a double-blind manner (A).
LR represents the left-right axis.
Each boundary is classified as a right-tilted boundary (orange) or left-tilted boundary (turquoise) (B).
The numbers of right-tilted (N~R~) and left-tilted (N~L~) boundaries were counted, and the chirality index was calculated using the formula shown at the bottom of B.

\clearpage

![](../figs/19/19.jpg)  
**Fig. 19. Chirality index is disrupted in *emc* mutants, which is rescued by *Myo31DF* overexpression.**
(A) Strip plots showing the chirality index of embryos with the genotypes indicated at the bottom.
 The mean values (orange line) and standard errors (black dotted line) are presented.
The grey dotted line indicates the 0 value of the chirality index.
"n" indicates the number of total boundaries analyzed, and "N" indicates the number of total embryos analyzed.
In the cases involving overexpression: *NP2432>Myo31DF::RFP* in *Myo31DF^K2^*, *Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>emc::GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-emc::GFP* by *NP2432*; *NP2432>Myo31DF::RFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>da*, wild-type overexpressing *UAS-da* by *NP2432*; *NP2432>da>Myo31DF::RFP*, wild-type overexpressing *UAS-da* and *UAS-Myo31DF::RFP* by *NP2432*.
(B) Matrix showing the statistical significances of the results shown in A.
For this analysis, Tukey-Kramer's multiple comparison test was done between all of the combinations shown in A.
The color of each square represents statistical significance between the genotypes presented at the left and at the bottom side of the matrix.
Yellow, orange, and red squares indicate p < 0.05, p < 0.01, and p < 0.001, respectively.
White square indicates p ≥ 0.05 (no significance).

\clearpage

![](../figs/20/20.jpg)  
**Fig. 20. *emc* and *da* are involved in the normal cell chirality formation**
(A-H) Typical examples of the apical cell boundaries in the hindgut epithelium of embryos with the indicated genotypes.
Wild-type (WT) (A), *Myo31DF^K2^* homozygote (B), *emc^AP6^* homozygote (C), *emc^AP6^* homozygote overexpressing *UAS-emc::GFP* (D), wild-type embryo overexpressing *UAS-da* (E), *Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP* (F), *emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP* (G), and wild-type embryo double-overexpressing *UAS-da* and *UAS-Myo31DF::RFP* (H) are shown.
 Overexpression was driven by *NP2432* in D, E, F, G, and H.
The apical cell boundaries were visualized by anti-*D*E-Cad antibody staining.
 Scale bars are 2.5 μm.

\clearpage

![](../figs/21/21.png)  
**Fig. 21. A model summarizing my thesis; Emc, Da, and Myo31DF are involved in the normal cell chirality formation.**
A model summarizing the revealed cascade involving Emc, Da, and Myo31DF in the cell chirality formation and counterclockwise rotation of the hindgut.
The repression of Da by Emc is required for the formation of the normal cell chirality.
Our group suggested that the normal cell chirality induced the wild-type rotation of the hindgut.
Myo31DF regulates the cell chirality under the control of or parallel to Emc/Da.

\clearpage

![](../figs/22/22.png)  
**Fig. 22. Chirality index sigmoidally correlates with the probability of the wild-type LR asymmetry of the hindgut in various embryos.**
Scatter plot showing the relationship between in chirality index and the probability of the normal LR laterality in the hindgut of embryos with the genotypes indicated at the bottom.
For the probability data, the phenotype frequencies of the LR asymmetry in the hindgut that calculated in Fig. 2, 8, and 11 are used.
For the chirality index data, the same values calculated in Fig. 19 are used.
Standard errors (horizontal lines) are presented according to Fig. 19.
The grey solid line and broken line indicate the linear fitting model and Hill equation fitting model, respectively.
Fitting parameters are shown in Table 3.
The numbers of embryos for the chirality index and for the probability of the LR asymmetry are shown in the bottom panel as *N~C.I.~* and *n~Prob.~*, respectively.
Correlation coefficient, coefficient of determination of the fitted linear model, and coefficient of determination of the fitted Hill equation model are shown at the upper right corner as R, Linear R^2^, and Hill R^2^, respectively.

\clearpage

![](../figs/23/23.png)  
**Fig. 23. Live chirality index sigmoidally correlates with the LR laterality of the individual *emc^AP6^* mutant embryos.**
Scatter plot showing the relationship between in live chirality index and the LR laterality of the hindgut in individual *emc^AP6^* mutant embryos.
The grey solid line and broken line indicate the linear fitting model and Hill equation fitting model, respectively.
Fitting parameters are shown in Table 3.
Correlation coefficient, coefficient of determination of the fitted linear model, and coefficient of determination of the fitted Hill equation model are shown at the upper right corner as R, Linear R^2^, and Hill R^2^, respectively.

\clearpage

![](../figs/24/24.jpg)  
**Fig. 24. The apical size is expanded in *emc*, but the expansion is not relevant to the cell chirality or LR asymmetry of the hindgut.**
(A) Strip plots showing the size of the apical cell surface in the hindgut epithelium of embryos with the genotypes indicated at the bottom.
The mean value (orange line) and standard error (black dotted line) are presented.
"n" and "N" indicate the number of total cells and total embryos analyzed, respectively.
In the cases involving overexpression: *NP2432>Myo31DF::RFP* in *Myo31DF^K2^*, *Myo31DF^K2^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>emc::GFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-emc::GFP* by *NP2432*; *NP2432>Myo31DF::RFP* in *emc^AP6^*, *emc^AP6^* homozygote overexpressing *UAS-Myo31DF::RFP* by *NP2432*; *NP2432>da*, wild-type overexpressing *UAS-da* by *NP2432*; *NP2432>da>Myo31DF::RFP*, wild-type overexpressing *UAS-da* and *UAS-Myo31DF::RFP* by *NP2432*.
 (B) Matrix showing the statistical significances of the results shown in A.
For this analysis, Tukey-Kramer's multiple comparison test was done between all of the combinations shown in A.
The color of each square represents statistical significance between the genotypes presented at the left and at the bottom side of the matrix.
Yellow, orange, and red squares indicate p < 0.05, p < 0.01, and p < 0.001, respectively.
White square indicates p ≥ 0.05 (no significance).

\clearpage

![](../figs/25/25.jpg)  
**Fig. 25. F-actin localization is not changed in *emc* mutant cells.**
(A-C) Mosaic hindgut epithelium composed of *emc^AP6^* mutant cells and the same mutant cells overexpressing *UAS-emc::GFP*, stained with phalloidin (A) and anti-GFP (B).
C is a merged image of A (shown in green) and B (shown in magenta).
A', B', and C' are high-magnification images of the regions indicated by white boxes in A, B, and C, respectively.
In A', asterisks indicate *emc* mutant cells overexpressing *UAS-emc::GFP*, and the arrowhead shows the apical region of the hindgut epithelial cells.
Scale bar indicates 20 μm (D) Graph showing the normalized signal intensity of phalloidin staining.
In the hindgut stained with rhodamine phalloidin, the normalized mean intensity of the rhodamine signal along the apical, lateral and basal cell boundaries in wild-type cells (white) and *emc^AP6^* homozygous cells (grey) was calculated.
"n" represents the numbers of cell boundaries analyzed.
Standard error is shown by black lines.
For the statistical analysis, a Welch's 2-sample t-test with a 5% significance level was used.

\clearpage

![](../figs/26/26.jpg)  
**Fig. 26. The number of the hindgut epithelial cells is not changed in *emc* mutants.**
(A-B) Typical examples of the hindgut stained with anti-Lamin C antibody.
Wild-type (A) and *emc^AP6^* homozygous mutant (B) were shown.
White dotted lines show the outline of the hindgut.
 Scale bar indicates 25 μm.
(C) Bar graph showing the cell number in wild-type (left) and *emc^AP6^* mutant (right).
Standard error is shown by black lines.
"N" indicates the number of total embryos analyzed.
For the statistical analysis, a Welch's 2-sample t-test with a 5% significance level was used.

\clearpage

![](../figs/27/27.png)  
**Fig. 27. A model where the cell chirality determines the rotational direction of the hindgut.**
(A) Schematic showing the possible model in which the cell chirality determines the rotational direction of the hindgut.
There are local minimums of the unstability with rotational direction to -90° (wild-type direction), 0° (non-lateral), and +90° (inverse direction), and this unstability is changed depending on the cell chirality.
Perturbations in individual embryos fluctuate, and it stochastically determines the rotational direction of the hindgut.
(B) In wild-type, the enough dextral cell chirality is generated, which induces -90°-biased unstability.
Therefore, the hindgut rotates to wild-type direction at 100% probability.
(B') In *emc* mutants, the cell chirality disappears and there are three local minimums of the unstability with rotational direction.
In this condition, the fluctuations of the cell chirality in individual embryos stochastically determine the rotational direction of the hindgut.

\clearpage

![](../figs/28/28.png)  
**Fig. 28. A model summarizing the novel role of E/Id regulatory system in the cell shape and cell size regulation revealed by this thesis.**
In this thesis, I revealed that *emc* and *da* is involved in the regulation of cell shape, such as the cell chirality and the apical size.
It was suggested that the cell shape regulated by *emc* is required for the proper control of organ shape.

\clearpage

<!--
| **Gene name**        | **Molecular Function**             | **Reference**                        |
| :------------------- | :--------------------------------- | :----------------------------------- |
| *extra macrochaetae* | HLH transcription factors          | [@Garrell1990]                       |
| *Notch*              | Notch signaling component          | [@Kidd1986; @Rice2015; @Wharton1985] |
| *Antennapedia*       | Homeobox transcription factor      | [@Laughon1986; @McGinnis1984]        |
| *Deformed*           | Homeobox transcription factor      | [@Regulski1985]                      |
| *engrailed*          | Homeobox transcription factor      | [@Kassis1986]                        |
| *cut*                | Homeobox transcription factor      | [@Blochlinger1988]                   |
| *bicoid*             | Homeobox transcription factor      | [@Berleth1988]                       |
| *daughterless*       | bHLH transcription factor          | [@Cronmiller1988]                    |
| *twist*              | bHLH transcription factor          | [@Thisse1988]                        |
| *single-minded*      | bHLH transcription factor          | [@Crews1988]                         |
| *taiman*             | bHLH transcription factor          | [@Bai2000]                           |
| *hairy*              | bHLH transcription corepressor     | [@Wainwright1992]                    |
| *hunchback*          | Zinc finger transcription factor   | [@Tautz1987]                         |
| *dorsal*             | NF-κB family transcription factor | [@Steward1987]                       |
-->

+----------------+------------------------------------+------------------+
|**Gene name**   | **Molecular Function**             | **Reference**    |
+================+====================================+==================+
|*extra*         | HLH transcription factor           | @Garrell1990     |
|*macrochaetae*  |                                    |                  |
+----------------+------------------------------------+------------------+
|*Notch*         | Notch signaling component          | @Kidd1986;       |
|                |                                    | @Rice2015;       |
|                |                                    | @Wharton1985;    |
+----------------+------------------------------------+------------------+
|*Antennapedia*  | Homeobox transcription factor      | @Laughon1986;    |
|                |                                    | @McGinnis1984;   |
+----------------+------------------------------------+------------------+
|*Deformed*      | Homeobox transcription factor      | @Regulski1985    |
+----------------+------------------------------------+------------------+
|*engrailed*     | Homeobox transcription factor      | @Kassis1986      |
+----------------+------------------------------------+------------------+
|*cut*           | Homeobox transcription factor      | @Blochlinger1988 |
+----------------+------------------------------------+------------------+
|*bicoid*        | Homeobox transcription factor      | @Berleth1988     |
+----------------+------------------------------------+------------------+
|*daughterless*  | bHLH transcription factor          | @Cronmiller1988  |
+----------------+------------------------------------+------------------+
|*twist*         | bHLH transcription factor          | @Thisse1988      |
+----------------+------------------------------------+------------------+
|*single-minded* | bHLH transcription factor          | @Crews1988       |
+----------------+------------------------------------+------------------+
|*taiman*        | bHLH transcription factor          | @Bai2000         |
+----------------+------------------------------------+------------------+
|*hairy*         | bHLH transcription corepressor     | @Wainwright1992  |
+----------------+------------------------------------+------------------+
|*hunchback*     | Zinc finger transcription factor   | @Tautz1987       |
+----------------+------------------------------------+------------------+
|*dorsal*        | NF-κB family transcription factor | @Steward1987     |
+----------------+------------------------------------+------------------+

**Table 1. Various *Drosophila* proteins have Gln-rich region.**
Table showing the genes including *opa* repeat in *Drosophila*.
The *Drosophila* gene that has Gln-rich region, the molecular function and the paper where the protein structure of the genes is reported are shown in the first, the second, and the third column, respectively.

\clearpage

| *emc* expression in the hindgut  | stage 11  | stage 12  | stage 14   |
| -------------------------------: | :-------- | :-------- | :--------- |
| +                                | 11        | 4         | 0          |
| +/-                              | 1         | 13        | 7          |
| -                                | 0         | 0         | 14         |
| **Total**                        | **12**    | **17**    | **21**     |

**Table 2. *emc* expresses in the hindgut during the hindgut rotation.**
The expression of *emc* was detected by *in situ* hybridization using an antisense probe for *emc* (see Figure 5).
The numbers of embryos showing *emc* signals in the hindgut primordium and hindgut are shown.
+, strong signal; +/-, weak signal; -, no signal.

\clearpage

| Model in Fig. 22         | Linear model             | Hill curve model                                         |
| :----------------------- | :----------------------  | :------------------------------------------------------- |
| Model                    | $\alpha \cdot x + \beta$ | $1 - \frac{(x + \alpha)^{n}}{\beta + (x + \alpha)^{n}}$  |
| $\alpha$                 | $-2.65542$               | $0.80126$                                                |
| $\beta$                  | $0.67360$                | $0.051899$                                               |
| $n$                      |                          | $20.8340$                                                |

| Model in Fig. 23         | Linear model             | Hill curve model                                         |
| :----------------------- | :----------------------  | :------------------------------------------------------- |
| Model                    | $\alpha \cdot x + \beta$ | $1 - \frac{(x + \alpha)^{n}}{\beta + (x + \alpha)^{n}}$  |
| $\alpha$                 | $-2.67589$               | $0.85350$                                                |
| $\beta$                  | $0.58362$                | $0.044309$                                               |
| $n$                      |                          | $21.843$                                                 |

<!--
+----------+--------------------------+---------------------------------------------------------+--------------------------+---------------------------------------------------------+
|          | Linear                   | Hill                                                    | Linear                   | Hill                                                    |
|          | model in                 | curve                                                   | model in                 | curve                                                   |
|          | Fig. 22                  | model in                                                | Fig. 23                  | model in                                                |
|          |                          | Fig. 22                                                 |                          | Fig. 23                                                 |
|          |                          |                                                         |                          |                                                         |
| Model    | $\alpha \cdot x + \beta$ | $1 - \frac{(x + \alpha)^{n}}{\beta + (x + \alpha)^{n}}$ | $\alpha \cdot x + \beta$ | $1 - \frac{(x + \alpha)^{n}}{\beta + (x + \alpha)^{n}}$ |
+==========+==========================+=========================================================+==========================+=========================================================+
| $\alpha$ | $-2.65542$               | $0.80126$                                               | $-2.67589$               | $0.85350$                                               |
+----------+--------------------------+---------------------------------------------------------+--------------------------+---------------------------------------------------------+
| $\beta$  | $0.67360$                | $0.051899$                                              | $0.58362$                | $0.044309$                                              |
+----------+--------------------------+---------------------------------------------------------+--------------------------+---------------------------------------------------------+
| $n$      |                          | $20.8340$                                               |                          | $21.843$                                                |
+----------+--------------------------+---------------------------------------------------------+--------------------------+---------------------------------------------------------+
-->

**Table 3. Fitting parameters in Fig. 22 and 23.**
The fitting equations and parameters used in Fig. 22 and Fig. 23 are shown.

\clearpage

\mbox{}\newpage

<!-- vim: set nowrap : -->
